diff --git a/.windsurfrules b/.windsurfrules
index ef6dc9c..18f196f 100644
--- a/.windsurfrules
+++ b/.windsurfrules
@@ -10,4 +10,6 @@ work with the existing design
 use existing componets
 do not run tests
 minial descricptions of work performed back to the user
-maintain consistancy with models in src\models\data use using properites
\ No newline at end of file
+maintain consistancy with models in src\models\data use using 
+Do not hard code game values. They are in src\models\data.
+minimize ruinning shell commands
