diff --git a/src/stores/index.ts b/src/stores/index.ts
index c1bf8ce..0e603bf 100644
--- a/src/stores/index.ts
+++ b/src/stores/index.ts
@@ -1,9 +1,17 @@
 import { useGameStore } from './gameStore';
 import { usePlayerStore } from './playerStore';
 import { useCardStore } from './cardStore';
+import { useSeasonStore } from './seasonStore';
+import { useJourneyStore } from './journeyStore';
+import { useChallenge } from './challengeStore';
+import { useLogStore } from './logStore';
 
 export {
   useGameStore,
   usePlayerStore,
-  useCardStore
+  useCardStore,
+  useSeasonStore,
+  useJourneyStore,
+  useChallenge,
+  useLogStore
 };
