diff --git a/src/data/enemy.js b/src/data/enemy.js
index d43b76e..c528526 100644
--- a/src/data/enemy.js
+++ b/src/data/enemy.js
@@ -34,18 +34,8 @@ class Enemy extends Unit {
   }
 
   // 执行行动
-  // @reutrn 
-  // {
-  //  endTurn: 表示是否结束此回合，否然act会被继续调用，默认为true
-  //  latency: 操作间隔，默认为800ms,
-  //  promise: 敌人自定义行动延时和异步操作，如果非空，则会被上层应用等待，默认为null
-  // }
   act(player) {
     // 子类需要实现具体逻辑
-    return {
-      endTurn: true,
-      latency: 800
-    }
   }
 }
 
