diff --git a/src/data/skill.js b/src/data/skill.js
index 06a658b..ad4e840 100644
--- a/src/data/skill.js
+++ b/src/data/skill.js
@@ -65,8 +65,7 @@ class Skill {
   // @param {Player} player: 玩家对象
   // @param {Enemy} enemy: 敌人对象
   // @param {Integer} stage: 此技能的使用阶段，默认为0，简单技能不需要考虑此参数。
-  // @return {boolean | null} bool: 是否完成使用技能，null: 技能使用失败
-  // 如果返回false，表明此技能是一个多阶段使用的技能，此方法会被连续调用且stage递增，直到返回true或null
+  // @return {boolean} 如果返回true，表示技能使用完成，否然，stage增加一，反复调用此技能。
   use(player, enemy, stage) {
     return true;
   }
