diff --git a/app/src/App.js b/app/src/App.js
index 1e58206..7bda2f3 100644
--- a/app/src/App.js
+++ b/app/src/App.js
@@ -1,6 +1,7 @@
 import React from "react";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import { AuthProvider } from "./context/AuthContext";
+import { TodoProvider } from "./context/TodoContext";
 import ProtectedRoute from "./components/auth/ProtectedRoute";
 import HomePage from "./pages/HomePage";
 import LoginForm from "./components/auth/LoginForm";
@@ -13,32 +14,34 @@ import "./App.css";
 function App() {
   return (
     <AuthProvider>
-      <Router>
-        <div className="App">
-          <Routes>
-            <Route path="/" element={<HomePage />} />
-            <Route path="/login" element={<LoginForm />} />
-            <Route path="/register" element={<RegisterForm />} />
-            <Route path="/verify-email" element={<EmailVerificationPage />} />
-            <Route
-              path="/login-success"
-              element={
-                <ProtectedRoute>
-                  <LoginSuccessPage />
-                </ProtectedRoute>
-              }
-            />
-            <Route
-              path="/todos"
-              element={
-                <ProtectedRoute>
-                  <TodoPage />
-                </ProtectedRoute>
-              }
-            />
-          </Routes>
-        </div>
-      </Router>
+      <TodoProvider>
+        <Router>
+          <div className="App">
+            <Routes>
+              <Route path="/" element={<HomePage />} />
+              <Route path="/login" element={<LoginForm />} />
+              <Route path="/register" element={<RegisterForm />} />
+              <Route path="/verify-email" element={<EmailVerificationPage />} />
+              <Route
+                path="/login-success"
+                element={
+                  <ProtectedRoute>
+                    <LoginSuccessPage />
+                  </ProtectedRoute>
+                }
+              />
+              <Route
+                path="/todos"
+                element={
+                  <ProtectedRoute>
+                    <TodoPage />
+                  </ProtectedRoute>
+                }
+              />
+            </Routes>
+          </div>
+        </Router>
+      </TodoProvider>
     </AuthProvider>
   );
 }
