diff --git a/src/models/types/player.ts b/src/models/types/player.ts
index 26f005a..4c77ea6 100644
--- a/src/models/types/player.ts
+++ b/src/models/types/player.ts
@@ -38,6 +38,7 @@ export interface PlayerState {
   hasCraftedLegendaryItem: boolean;
   uniqueCraftedItemsCount: number;
   companionLoyalty: { [companionId: string]: CompanionStatus };
+  wisdom: number;
 }
 
 export interface PlayerActionResult {
