diff --git a/src/App.jsx b/src/App.jsx
index 15745f8..4c94cfe 100644
--- a/src/App.jsx
+++ b/src/App.jsx
@@ -2,7 +2,6 @@ import React from 'react';
 import Wrapper from './components/Wrapper/Wrapper';
 import Header from './components/Header/Header';
 import Bio from './components/Bio/Bio';
-import Divider from './components/Divider/Divider';
 import Contact from './components/Contact/Contact';
 
 function App() {
@@ -10,7 +9,6 @@ function App() {
 		<Wrapper>
 			<Header />
 			<Bio />
-			<Divider />
 			<Contact />
 		</Wrapper>
 	);
