diff --git a/lib/src/screens/home_screen/home_screen.dart b/lib/src/screens/home_screen/home_screen.dart
index 72d17d2..a96ba4e 100644
--- a/lib/src/screens/home_screen/home_screen.dart
+++ b/lib/src/screens/home_screen/home_screen.dart
@@ -41,6 +41,7 @@ class _HomeScreenState extends State<HomeScreen> {
     _homeViewModel.removeListener(_onHomeViewModelChanged);
     _scrollController.removeListener(_onScroll);
     _scrollController.dispose();
+    _homeViewModel.dispose();
     super.dispose();
   }
 
