diff --git a/lib/src/screens/loading_screen/loading_route_provider.dart b/lib/src/screens/loading_screen/loading_route_provider.dart
index f52f106..75718ba 100644
--- a/lib/src/screens/loading_screen/loading_route_provider.dart
+++ b/lib/src/screens/loading_screen/loading_route_provider.dart
@@ -16,9 +16,7 @@ class LoadingRouteProvider implements RouteProvider {
   RouteBase getRoute() {
     return GoRoute(
       path: '/loading',
-      builder: (context, state) => LoadingScreen(
-        authViewModel: _authViewModel,
-      ),
+      builder: (context, state) => LoadingScreen(authViewModel: _authViewModel),
     );
   }
-}
\ No newline at end of file
+}
