diff --git a/lib/src/screens/login_screen/login_route_provider.dart b/lib/src/screens/login_screen/login_route_provider.dart
index 7207a97..33ca13d 100644
--- a/lib/src/screens/login_screen/login_route_provider.dart
+++ b/lib/src/screens/login_screen/login_route_provider.dart
@@ -16,9 +16,8 @@ class LoginRouteProvider implements RouteProvider {
   RouteBase getRoute() {
     return GoRoute(
       path: '/login',
-      builder: (context, state) => LoginScreen(
-        viewModel: _loginViewModelFactory.create(),
-      ),
+      builder: (context, state) =>
+          LoginScreen(viewModel: _loginViewModelFactory.create()),
     );
   }
-}
\ No newline at end of file
+}
