diff --git a/lib/src/screens/login_screen/login_screen.dart b/lib/src/screens/login_screen/login_screen.dart
index 20d46e4..1239cec 100644
--- a/lib/src/screens/login_screen/login_screen.dart
+++ b/lib/src/screens/login_screen/login_screen.dart
@@ -32,6 +32,7 @@ class _LoginScreenState extends State<LoginScreen> {
   @override
   void dispose() {
     _viewModel.removeListener(_onViewModelChanged);
+    _viewModel.dispose();
     super.dispose();
   }
 
