diff --git a/lib/src/screens/user_details/user_details_route_provider.dart b/lib/src/screens/user_details/user_details_route_provider.dart
index e09dde1..00804a2 100644
--- a/lib/src/screens/user_details/user_details_route_provider.dart
+++ b/lib/src/screens/user_details/user_details_route_provider.dart
@@ -25,4 +25,4 @@ class UserDetailsRouteProvider implements RouteProvider {
       },
     );
   }
-}
\ No newline at end of file
+}
