diff --git a/lib/src/screens/user_details/user_details_screen.dart b/lib/src/screens/user_details/user_details_screen.dart
index 372b95c..70c9419 100644
--- a/lib/src/screens/user_details/user_details_screen.dart
+++ b/lib/src/screens/user_details/user_details_screen.dart
@@ -27,6 +27,7 @@ class _UserDetailsScreenState extends State<UserDetailsScreen> {
   @override
   void dispose() {
     _viewModel.removeListener(_onViewModelChanged);
+    _viewModel.dispose();
     super.dispose();
   }
 
