diff --git a/lib/src/screens/user_details/user_details_viewmodel.dart b/lib/src/screens/user_details/user_details_viewmodel.dart
index 215afb4..df52f59 100644
--- a/lib/src/screens/user_details/user_details_viewmodel.dart
+++ b/lib/src/screens/user_details/user_details_viewmodel.dart
@@ -1,7 +1,7 @@
-import 'package:flutter/foundation.dart';
+import '../base_viewmodel.dart';
 
 /// ViewModel for handling user details screen logic.
-class UserDetailsViewModel extends ChangeNotifier {
+class UserDetailsViewModel extends DisposableViewModel {
   final String _login;
 
   UserDetailsViewModel(this._login);
@@ -26,4 +26,9 @@ class UserDetailsViewModel extends ChangeNotifier {
     _isLoading = false;
     notifyListeners();
   }
+
+  @override
+  void onDispose() {
+    _isLoading = false;
+  }
 }
