diff --git a/test/screens/home_screen/home_route_provider_test.mocks.dart b/test/screens/home_screen/home_route_provider_test.mocks.dart
index 533e972..074e5e2 100644
--- a/test/screens/home_screen/home_route_provider_test.mocks.dart
+++ b/test/screens/home_screen/home_route_provider_test.mocks.dart
@@ -129,6 +129,11 @@ class MockAuthViewModel extends _i1.Mock implements _i6.AuthViewModel {
     returnValueForMissingStub: null,
   );
 
+  @override
+  bool get disposed =>
+      (super.noSuchMethod(Invocation.getter(#disposed), returnValue: false)
+          as bool);
+
   @override
   bool get hasListeners =>
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
@@ -159,14 +164,8 @@ class MockAuthViewModel extends _i1.Mock implements _i6.AuthViewModel {
   );
 
   @override
-  void addListener(_i8.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#addListener, [listener]),
-    returnValueForMissingStub: null,
-  );
-
-  @override
-  void removeListener(_i8.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#removeListener, [listener]),
+  void onDispose() => super.noSuchMethod(
+    Invocation.method(#onDispose, []),
     returnValueForMissingStub: null,
   );
 
@@ -181,6 +180,18 @@ class MockAuthViewModel extends _i1.Mock implements _i6.AuthViewModel {
     Invocation.method(#notifyListeners, []),
     returnValueForMissingStub: null,
   );
+
+  @override
+  void addListener(_i8.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#addListener, [listener]),
+    returnValueForMissingStub: null,
+  );
+
+  @override
+  void removeListener(_i8.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#removeListener, [listener]),
+    returnValueForMissingStub: null,
+  );
 }
 
 /// A class which mocks [HomeViewModel].
@@ -214,6 +225,11 @@ class MockHomeViewModel extends _i1.Mock implements _i2.HomeViewModel {
       (super.noSuchMethod(Invocation.getter(#isEmpty), returnValue: false)
           as bool);
 
+  @override
+  bool get disposed =>
+      (super.noSuchMethod(Invocation.getter(#disposed), returnValue: false)
+          as bool);
+
   @override
   bool get hasListeners =>
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
@@ -252,6 +268,12 @@ class MockHomeViewModel extends _i1.Mock implements _i2.HomeViewModel {
     returnValueForMissingStub: null,
   );
 
+  @override
+  void onDispose() => super.noSuchMethod(
+    Invocation.method(#onDispose, []),
+    returnValueForMissingStub: null,
+  );
+
   @override
   void dispose() => super.noSuchMethod(
     Invocation.method(#dispose, []),
@@ -259,20 +281,20 @@ class MockHomeViewModel extends _i1.Mock implements _i2.HomeViewModel {
   );
 
   @override
-  void addListener(_i8.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#addListener, [listener]),
+  void notifyListeners() => super.noSuchMethod(
+    Invocation.method(#notifyListeners, []),
     returnValueForMissingStub: null,
   );
 
   @override
-  void removeListener(_i8.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#removeListener, [listener]),
+  void addListener(_i8.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#addListener, [listener]),
     returnValueForMissingStub: null,
   );
 
   @override
-  void notifyListeners() => super.noSuchMethod(
-    Invocation.method(#notifyListeners, []),
+  void removeListener(_i8.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#removeListener, [listener]),
     returnValueForMissingStub: null,
   );
 }
