diff --git a/test/screens/home_screen/home_screen_test.mocks.dart b/test/screens/home_screen/home_screen_test.mocks.dart
index 85bc667..fe87748 100644
--- a/test/screens/home_screen/home_screen_test.mocks.dart
+++ b/test/screens/home_screen/home_screen_test.mocks.dart
@@ -57,6 +57,11 @@ class MockHomeViewModel extends _i1.Mock implements _i2.HomeViewModel {
       (super.noSuchMethod(Invocation.getter(#isEmpty), returnValue: false)
           as bool);
 
+  @override
+  bool get disposed =>
+      (super.noSuchMethod(Invocation.getter(#disposed), returnValue: false)
+          as bool);
+
   @override
   bool get hasListeners =>
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
@@ -95,6 +100,12 @@ class MockHomeViewModel extends _i1.Mock implements _i2.HomeViewModel {
     returnValueForMissingStub: null,
   );
 
+  @override
+  void onDispose() => super.noSuchMethod(
+    Invocation.method(#onDispose, []),
+    returnValueForMissingStub: null,
+  );
+
   @override
   void dispose() => super.noSuchMethod(
     Invocation.method(#dispose, []),
@@ -102,20 +113,20 @@ class MockHomeViewModel extends _i1.Mock implements _i2.HomeViewModel {
   );
 
   @override
-  void addListener(_i5.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#addListener, [listener]),
+  void notifyListeners() => super.noSuchMethod(
+    Invocation.method(#notifyListeners, []),
     returnValueForMissingStub: null,
   );
 
   @override
-  void removeListener(_i5.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#removeListener, [listener]),
+  void addListener(_i5.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#addListener, [listener]),
     returnValueForMissingStub: null,
   );
 
   @override
-  void notifyListeners() => super.noSuchMethod(
-    Invocation.method(#notifyListeners, []),
+  void removeListener(_i5.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#removeListener, [listener]),
     returnValueForMissingStub: null,
   );
 }
@@ -150,6 +161,11 @@ class MockAuthViewModel extends _i1.Mock implements _i6.AuthViewModel {
     returnValueForMissingStub: null,
   );
 
+  @override
+  bool get disposed =>
+      (super.noSuchMethod(Invocation.getter(#disposed), returnValue: false)
+          as bool);
+
   @override
   bool get hasListeners =>
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
@@ -180,14 +196,8 @@ class MockAuthViewModel extends _i1.Mock implements _i6.AuthViewModel {
   );
 
   @override
-  void addListener(_i5.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#addListener, [listener]),
-    returnValueForMissingStub: null,
-  );
-
-  @override
-  void removeListener(_i5.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#removeListener, [listener]),
+  void onDispose() => super.noSuchMethod(
+    Invocation.method(#onDispose, []),
     returnValueForMissingStub: null,
   );
 
@@ -202,4 +212,16 @@ class MockAuthViewModel extends _i1.Mock implements _i6.AuthViewModel {
     Invocation.method(#notifyListeners, []),
     returnValueForMissingStub: null,
   );
+
+  @override
+  void addListener(_i5.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#addListener, [listener]),
+    returnValueForMissingStub: null,
+  );
+
+  @override
+  void removeListener(_i5.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#removeListener, [listener]),
+    returnValueForMissingStub: null,
+  );
 }
