diff --git a/test/screens/loading_screen/loading_screen_test.mocks.dart b/test/screens/loading_screen/loading_screen_test.mocks.dart
index 5e08cbd..167f095 100644
--- a/test/screens/loading_screen/loading_screen_test.mocks.dart
+++ b/test/screens/loading_screen/loading_screen_test.mocks.dart
@@ -53,6 +53,11 @@ class MockAuthViewModel extends _i1.Mock implements _i2.AuthViewModel {
     returnValueForMissingStub: null,
   );
 
+  @override
+  bool get disposed =>
+      (super.noSuchMethod(Invocation.getter(#disposed), returnValue: false)
+          as bool);
+
   @override
   bool get hasListeners =>
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
@@ -83,14 +88,8 @@ class MockAuthViewModel extends _i1.Mock implements _i2.AuthViewModel {
   );
 
   @override
-  void addListener(_i4.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#addListener, [listener]),
-    returnValueForMissingStub: null,
-  );
-
-  @override
-  void removeListener(_i4.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#removeListener, [listener]),
+  void onDispose() => super.noSuchMethod(
+    Invocation.method(#onDispose, []),
     returnValueForMissingStub: null,
   );
 
@@ -105,4 +104,16 @@ class MockAuthViewModel extends _i1.Mock implements _i2.AuthViewModel {
     Invocation.method(#notifyListeners, []),
     returnValueForMissingStub: null,
   );
+
+  @override
+  void addListener(_i4.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#addListener, [listener]),
+    returnValueForMissingStub: null,
+  );
+
+  @override
+  void removeListener(_i4.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#removeListener, [listener]),
+    returnValueForMissingStub: null,
+  );
 }
