diff --git a/test/screens/login_screen/login_route_provider_test.mocks.dart b/test/screens/login_screen/login_route_provider_test.mocks.dart
index 85d8b83..e072a49 100644
--- a/test/screens/login_screen/login_route_provider_test.mocks.dart
+++ b/test/screens/login_screen/login_route_provider_test.mocks.dart
@@ -70,11 +70,22 @@ class MockLoginViewModel extends _i1.Mock implements _i2.LoginViewModel {
       (super.noSuchMethod(Invocation.getter(#isAuthorized), returnValue: false)
           as bool);
 
+  @override
+  bool get disposed =>
+      (super.noSuchMethod(Invocation.getter(#disposed), returnValue: false)
+          as bool);
+
   @override
   bool get hasListeners =>
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
           as bool);
 
+  @override
+  void onDispose() => super.noSuchMethod(
+    Invocation.method(#onDispose, []),
+    returnValueForMissingStub: null,
+  );
+
   @override
   _i4.Future<void> login() =>
       (super.noSuchMethod(
@@ -85,26 +96,26 @@ class MockLoginViewModel extends _i1.Mock implements _i2.LoginViewModel {
           as _i4.Future<void>);
 
   @override
-  void addListener(_i5.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#addListener, [listener]),
+  void dispose() => super.noSuchMethod(
+    Invocation.method(#dispose, []),
     returnValueForMissingStub: null,
   );
 
   @override
-  void removeListener(_i5.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#removeListener, [listener]),
+  void notifyListeners() => super.noSuchMethod(
+    Invocation.method(#notifyListeners, []),
     returnValueForMissingStub: null,
   );
 
   @override
-  void dispose() => super.noSuchMethod(
-    Invocation.method(#dispose, []),
+  void addListener(_i5.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#addListener, [listener]),
     returnValueForMissingStub: null,
   );
 
   @override
-  void notifyListeners() => super.noSuchMethod(
-    Invocation.method(#notifyListeners, []),
+  void removeListener(_i5.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#removeListener, [listener]),
     returnValueForMissingStub: null,
   );
 }
