diff --git a/test/screens/login_screen/login_screen_test.mocks.dart b/test/screens/login_screen/login_screen_test.mocks.dart
index 172107e..6656c60 100644
--- a/test/screens/login_screen/login_screen_test.mocks.dart
+++ b/test/screens/login_screen/login_screen_test.mocks.dart
@@ -157,6 +157,11 @@ class MockAuthViewModel extends _i1.Mock implements _i6.AuthViewModel {
     returnValueForMissingStub: null,
   );
 
+  @override
+  bool get disposed =>
+      (super.noSuchMethod(Invocation.getter(#disposed), returnValue: false)
+          as bool);
+
   @override
   bool get hasListeners =>
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
@@ -187,14 +192,8 @@ class MockAuthViewModel extends _i1.Mock implements _i6.AuthViewModel {
   );
 
   @override
-  void addListener(_i7.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#addListener, [listener]),
-    returnValueForMissingStub: null,
-  );
-
-  @override
-  void removeListener(_i7.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#removeListener, [listener]),
+  void onDispose() => super.noSuchMethod(
+    Invocation.method(#onDispose, []),
     returnValueForMissingStub: null,
   );
 
@@ -209,6 +208,18 @@ class MockAuthViewModel extends _i1.Mock implements _i6.AuthViewModel {
     Invocation.method(#notifyListeners, []),
     returnValueForMissingStub: null,
   );
+
+  @override
+  void addListener(_i7.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#addListener, [listener]),
+    returnValueForMissingStub: null,
+  );
+
+  @override
+  void removeListener(_i7.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#removeListener, [listener]),
+    returnValueForMissingStub: null,
+  );
 }
 
 /// A class which mocks [LoginViewModel].
@@ -229,11 +240,22 @@ class MockLoginViewModel extends _i1.Mock implements _i8.LoginViewModel {
       (super.noSuchMethod(Invocation.getter(#isAuthorized), returnValue: false)
           as bool);
 
+  @override
+  bool get disposed =>
+      (super.noSuchMethod(Invocation.getter(#disposed), returnValue: false)
+          as bool);
+
   @override
   bool get hasListeners =>
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
           as bool);
 
+  @override
+  void onDispose() => super.noSuchMethod(
+    Invocation.method(#onDispose, []),
+    returnValueForMissingStub: null,
+  );
+
   @override
   _i3.Future<void> login() =>
       (super.noSuchMethod(
@@ -244,26 +266,26 @@ class MockLoginViewModel extends _i1.Mock implements _i8.LoginViewModel {
           as _i3.Future<void>);
 
   @override
-  void addListener(_i7.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#addListener, [listener]),
+  void dispose() => super.noSuchMethod(
+    Invocation.method(#dispose, []),
     returnValueForMissingStub: null,
   );
 
   @override
-  void removeListener(_i7.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#removeListener, [listener]),
+  void notifyListeners() => super.noSuchMethod(
+    Invocation.method(#notifyListeners, []),
     returnValueForMissingStub: null,
   );
 
   @override
-  void dispose() => super.noSuchMethod(
-    Invocation.method(#dispose, []),
+  void addListener(_i7.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#addListener, [listener]),
     returnValueForMissingStub: null,
   );
 
   @override
-  void notifyListeners() => super.noSuchMethod(
-    Invocation.method(#notifyListeners, []),
+  void removeListener(_i7.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#removeListener, [listener]),
     returnValueForMissingStub: null,
   );
 }
