diff --git a/test/screens/login_screen/login_viewmodel_test.mocks.dart b/test/screens/login_screen/login_viewmodel_test.mocks.dart
index 704a602..62f0a03 100644
--- a/test/screens/login_screen/login_viewmodel_test.mocks.dart
+++ b/test/screens/login_screen/login_viewmodel_test.mocks.dart
@@ -156,6 +156,11 @@ class MockAuthViewModel extends _i1.Mock implements _i6.AuthViewModel {
     returnValueForMissingStub: null,
   );
 
+  @override
+  bool get disposed =>
+      (super.noSuchMethod(Invocation.getter(#disposed), returnValue: false)
+          as bool);
+
   @override
   bool get hasListeners =>
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
@@ -186,14 +191,8 @@ class MockAuthViewModel extends _i1.Mock implements _i6.AuthViewModel {
   );
 
   @override
-  void addListener(_i7.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#addListener, [listener]),
-    returnValueForMissingStub: null,
-  );
-
-  @override
-  void removeListener(_i7.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#removeListener, [listener]),
+  void onDispose() => super.noSuchMethod(
+    Invocation.method(#onDispose, []),
     returnValueForMissingStub: null,
   );
 
@@ -208,4 +207,16 @@ class MockAuthViewModel extends _i1.Mock implements _i6.AuthViewModel {
     Invocation.method(#notifyListeners, []),
     returnValueForMissingStub: null,
   );
+
+  @override
+  void addListener(_i7.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#addListener, [listener]),
+    returnValueForMissingStub: null,
+  );
+
+  @override
+  void removeListener(_i7.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#removeListener, [listener]),
+    returnValueForMissingStub: null,
+  );
 }
