diff --git a/test/screens/user_details/user_details_route_provider_test.mocks.dart b/test/screens/user_details/user_details_route_provider_test.mocks.dart
index 3833435..0ac4243 100644
--- a/test/screens/user_details/user_details_route_provider_test.mocks.dart
+++ b/test/screens/user_details/user_details_route_provider_test.mocks.dart
@@ -79,6 +79,11 @@ class MockUserDetailsViewModel extends _i1.Mock
       (super.noSuchMethod(Invocation.getter(#isLoading), returnValue: false)
           as bool);
 
+  @override
+  bool get disposed =>
+      (super.noSuchMethod(Invocation.getter(#disposed), returnValue: false)
+          as bool);
+
   @override
   bool get hasListeners =>
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
@@ -94,14 +99,8 @@ class MockUserDetailsViewModel extends _i1.Mock
           as _i5.Future<void>);
 
   @override
-  void addListener(_i6.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#addListener, [listener]),
-    returnValueForMissingStub: null,
-  );
-
-  @override
-  void removeListener(_i6.VoidCallback? listener) => super.noSuchMethod(
-    Invocation.method(#removeListener, [listener]),
+  void onDispose() => super.noSuchMethod(
+    Invocation.method(#onDispose, []),
     returnValueForMissingStub: null,
   );
 
@@ -116,4 +115,16 @@ class MockUserDetailsViewModel extends _i1.Mock
     Invocation.method(#notifyListeners, []),
     returnValueForMissingStub: null,
   );
+
+  @override
+  void addListener(_i6.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#addListener, [listener]),
+    returnValueForMissingStub: null,
+  );
+
+  @override
+  void removeListener(_i6.VoidCallback? listener) => super.noSuchMethod(
+    Invocation.method(#removeListener, [listener]),
+    returnValueForMissingStub: null,
+  );
 }
