diff --git a/client/src/components/PlayerCharacter.vue b/client/src/components/PlayerCharacter.vue
index 101c0ea..5693f28 100644
--- a/client/src/components/PlayerCharacter.vue
+++ b/client/src/components/PlayerCharacter.vue
@@ -71,6 +71,7 @@ const handleKeyDown = (e: KeyboardEvent) => {
   
   if (key === 'e') {
     // Use for interaction only
+    clearPressedKeys();
     gameStore.emitEvent('player-interaction');
     return;
   }
