diff --git a/src/models/data/resources.js b/src/models/data/resources.js
index 395b3b3..3ebdfa7 100644
--- a/src/models/data/resources.js
+++ b/src/models/data/resources.js
@@ -109,5 +109,14 @@ export const resources = [
     description: 'Wooden sticks inscribed with Ogham symbols, used to decipher magical writings.',
     image: 'ogham_sticks.jpg',
     preferredBy: ['deer']
+  },
+  {
+    id: 'ancient_crystal',
+    name: 'Ancient Crystal',
+    effect: 'Amplifies spiritual energy',
+    type: 'mystic',
+    description: 'A valuable crystal with properties that can amplify magical energies.',
+    image: 'ancient_crystal.jpg',
+    preferredBy: ['owl']
   }
 ]
