diff --git a/src/models/enums/cardTypes.ts b/src/models/enums/cardTypes.ts
index c64093a..75def40 100644
--- a/src/models/enums/cardTypes.ts
+++ b/src/models/enums/cardTypes.ts
@@ -4,7 +4,8 @@ export enum CardType {
   ANIMAL_COMPANION = 'ANIMAL_COMPANION',
   RESOURCE = 'RESOURCE',
   CRAFTED_ITEM = 'CRAFTED_ITEM',
-  SEASON = 'SEASON'
+  SEASON = 'SEASON',
+  ACTION = 'ACTION'
 }
 
 export enum ChallengeType {
