diff --git a/src/components/core/GameCard.vue b/src/components/core/GameCard.vue
index 338a82a..7c692ae 100644
--- a/src/components/core/GameCard.vue
+++ b/src/components/core/GameCard.vue
@@ -42,9 +42,10 @@ export default defineComponent({
       default: ''
     },
     cardType: {
-      type: String as PropType<CardType>,
+      type: [String, Object] as PropType<CardType>,
       required: true,
-      validator: (value: string) => {
+      validator: (value: string | CardType) => {
+        // Allow both string values and enum values to be passed
         return Object.values(CardType).includes(value as CardType);
       }
     },
