diff --git a/app/pages/playground/index.vue b/app/pages/playground/index.vue
index 22eae31..f7d36c1 100644
--- a/app/pages/playground/index.vue
+++ b/app/pages/playground/index.vue
@@ -1,4 +1,5 @@
 <script setup>
+// no need to manually register playground pages, they are automatically registered
 const router = useRouter();
 const routes = router.getRoutes();
 const items = routes.filter((route) => route.path.startsWith("/playground/"));
