diff --git a/app/panel/src/services/permission-service.ts b/app/panel/src/services/permission-service.ts
index bbbc513..5199690 100644
--- a/app/panel/src/services/permission-service.ts
+++ b/app/panel/src/services/permission-service.ts
@@ -3,6 +3,7 @@ export enum Permission {
   OPEN_LOCKER = 'open_locker',
   BULK_OPEN = 'bulk_open',
   BLOCK_LOCKER = 'block_locker',
+  MANAGE_LOCKERS = 'manage_lockers',
   MANAGE_VIP = 'manage_vip',
   MANAGE_MASTER_PIN = 'manage_master_pin',
   VIEW_EVENTS = 'view_events',
@@ -20,6 +21,7 @@ export class PermissionService {
       Permission.OPEN_LOCKER,
       Permission.BULK_OPEN,
       Permission.BLOCK_LOCKER,
+      Permission.MANAGE_LOCKERS,
       Permission.MANAGE_VIP,
       Permission.MANAGE_MASTER_PIN,
       Permission.VIEW_EVENTS,
