diff --git a/test/screens/login_screen/login_screen_test.mocks.dart b/test/screens/login_screen/login_screen_test.mocks.dart
index 6656c60..8ce7ee8 100644
--- a/test/screens/login_screen/login_screen_test.mocks.dart
+++ b/test/screens/login_screen/login_screen_test.mocks.dart
@@ -250,12 +250,6 @@ class MockLoginViewModel extends _i1.Mock implements _i8.LoginViewModel {
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
           as bool);
 
-  @override
-  void onDispose() => super.noSuchMethod(
-    Invocation.method(#onDispose, []),
-    returnValueForMissingStub: null,
-  );
-
   @override
   _i3.Future<void> login() =>
       (super.noSuchMethod(
@@ -265,6 +259,12 @@ class MockLoginViewModel extends _i1.Mock implements _i8.LoginViewModel {
           )
           as _i3.Future<void>);
 
+  @override
+  void onDispose() => super.noSuchMethod(
+    Invocation.method(#onDispose, []),
+    returnValueForMissingStub: null,
+  );
+
   @override
   void dispose() => super.noSuchMethod(
     Invocation.method(#dispose, []),
