diff --git a/webview-ui/vite.config.ts b/webview-ui/vite.config.ts
index c185a9f..47d00fa 100644
--- a/webview-ui/vite.config.ts
+++ b/webview-ui/vite.config.ts
@@ -7,7 +7,7 @@ export default defineConfig({
   plugins: [preact()],
   root: '.',
   build: {
-    outDir: resolve(__dirname, '../media/ui'),
+    outDir: resolve(__dirname, '../out/ui'),
     emptyOutDir: true,
     sourcemap: false,
     rollupOptions: {
