diff --git a/src/modules/build/build.module.ts b/src/modules/build/build.module.ts
index a6ca8a0..e5d0056 100644
--- a/src/modules/build/build.module.ts
+++ b/src/modules/build/build.module.ts
@@ -3,9 +3,10 @@ import { Module } from '@nestjs/common';
 import { BuildCommand } from './commands/build.command';
 import { InteractiveService } from './services/interactive.service';
 import { CollectionService } from './services/collection.service';
+import { TransformationService } from './services/transformation.service';
 
 @Module({
-  providers: [BuildCommand, InteractiveService, CollectionService],
-  exports: [BuildCommand, InteractiveService, CollectionService],
+  providers: [BuildCommand, InteractiveService, CollectionService, TransformationService],
+  exports: [BuildCommand, InteractiveService, CollectionService, TransformationService],
 })
 export class BuildModule {}
\ No newline at end of file
