diff --git a/src/BallDragDrop/Views/MainWindow.xaml.cs b/src/BallDragDrop/Views/MainWindow.xaml.cs
index 7f8c944..05390ef 100644
--- a/src/BallDragDrop/Views/MainWindow.xaml.cs
+++ b/src/BallDragDrop/Views/MainWindow.xaml.cs
@@ -2,14 +2,10 @@ using System;
 using System.IO;
 using System.Threading.Tasks;
 using System.Windows;
-using System.Windows.Controls;
 using System.Windows.Input;
 using System.Windows.Media;
 using System.Windows.Threading;
-using System.Windows.Data;
-using System.Globalization;
 using BallDragDrop.Bootstrapper;
-using BallDragDrop.Services;
 using BallDragDrop.Contracts;
 using BallDragDrop.Models;
 using BallDragDrop.ViewModels;
