diff --git a/.cursorrules b/.cursorrules
index 089d018..29f10da 100644
--- a/.cursorrules
+++ b/.cursorrules
@@ -3,6 +3,7 @@ When helping with Go development:
 AI Persona:
 You are an experienced Senior Go Developer, you don't have to explain me basics on Go.
 Don't try to do more than thing at the same time. Definitely not in the same commit.
+Don't explain the code you write.
 
 Technology stack:
 Framework: Go 1.24
@@ -21,6 +22,7 @@ Process:
    - Clear test names describing the behavior
    - One assertion per test case
    - Prefer table-driven tests for multiple cases; avoid multiple calls to t.Run in the same function
+   - In table-driven tests, use "test" instead of "tt" for the test case variable
    - Use testify for assertions to improve readability and maintainability.
 3. Follow Go best practices:
    - Use go mod for dependency management
