commit 5907aabe5d1d020eed01f1867b0b9c30450669c9
Author:     Daniel Kelly <me@danielkelly.io>
AuthorDate: Wed Jul 2 16:26:27 2025 -0500
Commit:     Daniel Kelly <me@danielkelly.io>
CommitDate: Wed Jul 2 16:27:40 2025 -0500

    fix: component rules

diff --git a/.cursor/rules/pages.mdc b/.cursor/rules/pages.mdc
new file mode 100644
index 0000000..61cc7fa
--- /dev/null
+++ b/.cursor/rules/pages.mdc
@@ -0,0 +1,9 @@
+---
+description: 
+globs: 
+alwaysApply: false
+---
+* Prefer to fetch data at the page level most of the time. Ask me if you think you should do differently, based on the use case.
+* When possible, rely on the type safe return (typed by Nuxt) of useFetch and don't manually type the response. For example:
+  * DO THIS: `const {data} = await useFetch('/api/posts') // data is automatically typed`
+  * NOT THIS: `const {data} = await useFetch<Post>('/api/posts') // manually typing response is bad and leads to maintainence issues`
\ No newline at end of file


commit 089bfa4d17aed24c7a930f432fc34b0884cd0f91
Author:     Daniel Kelly <me@danielkelly.io>
AuthorDate: Mon Jul 7 14:30:36 2025 -0500
Commit:     Daniel Kelly <me@danielkelly.io>
CommitDate: Mon Jul 7 14:30:36 2025 -0500

    feat: update rules for common errors and database interactions

diff --git a/.cursor/rules/pages.mdc b/.cursor/rules/pages.mdc
index 61cc7fa..d595a14 100644
--- a/.cursor/rules/pages.mdc
+++ b/.cursor/rules/pages.mdc
@@ -6,4 +6,5 @@ alwaysApply: false
 * Prefer to fetch data at the page level most of the time. Ask me if you think you should do differently, based on the use case.
 * When possible, rely on the type safe return (typed by Nuxt) of useFetch and don't manually type the response. For example:
   * DO THIS: `const {data} = await useFetch('/api/posts') // data is automatically typed`
-  * NOT THIS: `const {data} = await useFetch<Post>('/api/posts') // manually typing response is bad and leads to maintainence issues`
\ No newline at end of file
+  * NOT THIS: `const {data} = await useFetch<Post>('/api/posts') // manually typing response is bad and leads to maintainence issues`
+* Break down pages into smaller components.
\ No newline at end of file


