commit 9ed6010d9bb94f0d45608975b4cfbb72ceecd7de
Author:     Daniel Kelly <me@danielkelly.io>
AuthorDate: Wed Jul 2 02:55:29 2025 -0500
Commit:     Daniel Kelly <me@danielkelly.io>
CommitDate: Wed Jul 2 02:55:29 2025 -0500

    legacy components rules

diff --git a/.cursor/rules/z-components.mdc b/.cursor/rules/z-components.mdc
new file mode 100644
index 0000000..3af8ef9
--- /dev/null
+++ b/.cursor/rules/z-components.mdc
@@ -0,0 +1,11 @@
+---
+description: 
+globs: 
+alwaysApply: false
+---
+* When creating a brand new stand alone component (that's not a child/tightly coupled component), use the `nlx hygen component new --name [ComponentNameHere]` from package.json* 
+* ALWAYS read the created files after running the hygen command.
+* When creating template refs always name them with the `El` suffix. For example:
+  * `const textareaEl = useTemplateRef('textareaEl')`  
+  * Always use `useTemplateRef('[ref identifier here]')` to define template refs (not `ref()`)
+  * ShadCN has components like `Textarea` that seem like native components but are not. Remember to account for that.
