diff --git a/internal/domain/user/.cursor/rules/user.mdc b/internal/domain/user/.cursor/rules/user.mdc
index baf5941..ffae78a 100644
--- a/internal/domain/user/.cursor/rules/user.mdc
+++ b/internal/domain/user/.cursor/rules/user.mdc
@@ -16,11 +16,6 @@ This rules describes about **user** entity and it's behaviour
 - Role: Enumerated string of "admin", "member", default("member") - constructor input
 - CipherText: CipherText - Default: "", secret
 - Status: Enumerated string of "active", "inactive", default("active")
-- CreatedAt: time.Time - Default: now
-- UpdatedAt: time.Time - Default: now
-- DeletedAt: time.Time - Default: nil, secret
-- IsDeleted: boolean - Default : false, secret
-
 ---
 
 
