diff --git a/src/GameApp.vue b/src/GameApp.vue
index 8e81363..bb17f8e 100644
--- a/src/GameApp.vue
+++ b/src/GameApp.vue
@@ -11,7 +11,6 @@
       v-else-if="gameState.gameStage === 'battle'"
       :player="gameState.player"
       :enemy="gameState.enemy"
-      :is-control-disabled="gameState.controlDisableCount > 0"
       :is-player-turn="!gameState.isEnemyTurn"
       :level="gameState.battleCount"
     />
