diff --git a/src/components/VoiceRecorder.tsx b/src/components/VoiceRecorder.tsx
index cc9e262..d03af69 100644
--- a/src/components/VoiceRecorder.tsx
+++ b/src/components/VoiceRecorder.tsx
@@ -1,8 +1,8 @@
 'use client';
 
 import { useState } from 'react';
-import { useDeepgram } from '../contexts/DeepgramContext';
-import { addDocument } from '../lib/firebaseUtils';
+import { useDeepgram } from '../lib/contexts/DeepgramContext';
+import { addDocument } from '../lib/firebase/firebaseUtils';
 
 export default function VoiceRecorder() {
   const [isRecording, setIsRecording] = useState(false);
