diff --git a/src/app/routes/__root.tsx b/src/app/routes/__root.tsx
index 32ca5a9..1a05c21 100644
--- a/src/app/routes/__root.tsx
+++ b/src/app/routes/__root.tsx
@@ -1,15 +1,24 @@
 import { Outlet, createRootRoute } from '@tanstack/react-router';
 import { TanStackRouterDevtools } from '@tanstack/react-router-devtools';
 
+import { GNB } from '@/widgets/gnb';
+
 export const Route = createRootRoute({
   component: RootComponent,
 });
 
 function RootComponent() {
   return (
-    <>
-      <Outlet />
-      <TanStackRouterDevtools />
-    </>
+    <div className="flex h-screen w-screen flex-col items-center justify-center">
+      <div className="container mx-auto w-full flex-1">
+        <div className="mt-10 flex flex-col items-center justify-center md:mx-20 lg:mx-40">
+          <GNB />
+          <div className="mt-10 w-full p-10">
+            <Outlet />
+          </div>
+        </div>
+        <TanStackRouterDevtools />
+      </div>
+    </div>
   );
 }
