diff --git a/src/app/routes/index.tsx b/src/app/routes/index.tsx
index 918bbf0..26c02e7 100644
--- a/src/app/routes/index.tsx
+++ b/src/app/routes/index.tsx
@@ -1,9 +1,15 @@
 import { createFileRoute } from '@tanstack/react-router';
 
+import { TypographyH4 } from '@/shared/components/shadcn-ui/typography';
+
 export const Route = createFileRoute('/')({
   component: RouteComponent,
 });
 
 function RouteComponent() {
-  return <div>Hello "/"!</div>;
+  return (
+    <div>
+      <TypographyH4>Home</TypographyH4>
+    </div>
+  );
 }
