diff --git a/src/app/routes/posts.$postId.tsx b/src/app/routes/posts.$postId.tsx
new file mode 100644
index 0000000..6ec24bf
--- /dev/null
+++ b/src/app/routes/posts.$postId.tsx
@@ -0,0 +1,16 @@
+import { createFileRoute } from '@tanstack/react-router';
+
+export const Route = createFileRoute('/posts/$postId')({
+  component: RouteComponent,
+  loader: async ({ params }) => {
+    return {
+      postId: params.postId,
+    };
+  },
+});
+
+function RouteComponent() {
+  const { postId } = Route.useLoaderData();
+
+  return <div>Hello /posts/{postId}</div>;
+}
