diff --git a/src/shared/components/shadcn-ui/sonner.tsx b/src/shared/components/shadcn-ui/sonner.tsx
new file mode 100644
index 0000000..cdc8813
--- /dev/null
+++ b/src/shared/components/shadcn-ui/sonner.tsx
@@ -0,0 +1,25 @@
+import { type CSSProperties } from 'react';
+
+import { useTheme } from 'next-themes';
+import { Toaster as Sonner, type ToasterProps } from 'sonner';
+
+const Toaster = ({ ...props }: ToasterProps) => {
+  const { theme = 'system' } = useTheme();
+
+  return (
+    <Sonner
+      theme={theme as ToasterProps['theme']}
+      className="toaster group"
+      style={
+        {
+          '--normal-bg': 'var(--popover)',
+          '--normal-text': 'var(--popover-foreground)',
+          '--normal-border': 'var(--border)',
+        } as CSSProperties
+      }
+      {...props}
+    />
+  );
+};
+
+export { Toaster };
