diff --git a/src/components/SkillsHand.vue b/src/components/SkillsHand.vue
index 6e5d162..08e60ac 100644
--- a/src/components/SkillsHand.vue
+++ b/src/components/SkillsHand.vue
@@ -16,6 +16,7 @@
         :player-mana="(player && player.mana != null) ? player.mana : Infinity"
         :can-click="draggable || (isPlayerTurn && !isControlDisabled && canUseSkill(skill))"
         :suppress-activation-animation-on-click="true"
+        :auto-register-in-registry="false"
         :ref="el => setCardRef(el, skill.uniqueID)"
         @skill-card-clicked="onSkillCardClicked"
       />
