diff --git a/src/data/player.js b/src/data/player.js
index 1d65ac5..9d719d4 100644
--- a/src/data/player.js
+++ b/src/data/player.js
@@ -92,15 +92,18 @@ export class Player extends Unit {
     this.maxSkills = 20; // 玩家拥有的总技能上限
     this.cultivatedSkills = []; // 已培养技能（顺序即为战斗中的默认顺序）
     // 场上技能与战斗列表
-    this.skills = []; // 场上技能。在战斗开始前由 cultivatedSkills 深拷贝生成，在战斗结束后清空。
+    this.skills = []; // 初始场上技能。在战斗开始前由 cultivatedSkills 深拷贝生成，在战斗结束后清空。
     this.frontierSkills = []; // 前台技能列表，玩家在当前回合可以使用的技能
     this.backupSkills = []; // 后备技能列表，用于存储暂时不可用的技能
+    this.burntSkills = []; // 坟地技能列表，存放被焚毁的技能（战斗中完全消耗掉的技能）
     this.maxFrontierSkills = 10; // 最大前台技能数量
     this.drawFrontierSkills = 4; // 每回合抽取前台技能数量
     // effects 由 Unit 初始化
     this.leino = ['normal']; // 灵脉列表，可以包含normal, fire, wind, wood, earth, water, thunder, light, dark
     this.abilities = []; // 玩家能力列表
 
+    this.uniqueID = 'playeruniqueid'; // 玩家唯一ID（用于动画同步等）
+
     // 属性修正器管线（按顺序应用）
     this.modifiers = [];
   }
