diff --git a/test/services/ferry_client_service_test.mocks.dart b/test/services/ferry_client_service_test.mocks.dart
index 194b53a..b4d7312 100644
--- a/test/services/ferry_client_service_test.mocks.dart
+++ b/test/services/ferry_client_service_test.mocks.dart
@@ -5,8 +5,9 @@
 // ignore_for_file: no_leading_underscores_for_library_prefixes
 import 'dart:async' as _i3;
 
-import 'package:gh3/src/services/token_storage.dart' as _i2;
+import 'package:gh3/src/services/auth_service.dart' as _i2;
 import 'package:mockito/mockito.dart' as _i1;
+import 'package:mockito/src/dummies.dart' as _i4;
 
 // ignore_for_file: type=lint
 // ignore_for_file: avoid_redundant_argument_values
@@ -22,35 +23,55 @@ import 'package:mockito/mockito.dart' as _i1;
 // ignore_for_file: camel_case_types
 // ignore_for_file: subtype_of_sealed_class
 
-/// A class which mocks [ITokenStorage].
+/// A class which mocks [AuthService].
 ///
 /// See the documentation for Mockito's code generation for more information.
-class MockITokenStorage extends _i1.Mock implements _i2.ITokenStorage {
-  MockITokenStorage() {
+class MockAuthService extends _i1.Mock implements _i2.AuthService {
+  MockAuthService() {
     _i1.throwOnMissingStub(this);
   }
 
   @override
-  _i3.Future<String?> getToken() =>
-      (super.noSuchMethod(
-            Invocation.method(#getToken, []),
-            returnValue: _i3.Future<String?>.value(),
-          )
-          as _i3.Future<String?>);
+  bool get isLoggedIn =>
+      (super.noSuchMethod(Invocation.getter(#isLoggedIn), returnValue: false)
+          as bool);
 
   @override
-  _i3.Future<void> saveToken(String? token) =>
+  _i3.Future<void> init() =>
       (super.noSuchMethod(
-            Invocation.method(#saveToken, [token]),
+            Invocation.method(#init, []),
             returnValue: _i3.Future<void>.value(),
             returnValueForMissingStub: _i3.Future<void>.value(),
           )
           as _i3.Future<void>);
 
   @override
-  _i3.Future<void> deleteToken() =>
+  _i3.Future<String> login() =>
+      (super.noSuchMethod(
+            Invocation.method(#login, []),
+            returnValue: _i3.Future<String>.value(
+              _i4.dummyValue<String>(this, Invocation.method(#login, [])),
+            ),
+          )
+          as _i3.Future<String>);
+
+  @override
+  _i3.Future<String> loginWithDeviceCode(String? deviceCode) =>
+      (super.noSuchMethod(
+            Invocation.method(#loginWithDeviceCode, [deviceCode]),
+            returnValue: _i3.Future<String>.value(
+              _i4.dummyValue<String>(
+                this,
+                Invocation.method(#loginWithDeviceCode, [deviceCode]),
+              ),
+            ),
+          )
+          as _i3.Future<String>);
+
+  @override
+  _i3.Future<void> logout() =>
       (super.noSuchMethod(
-            Invocation.method(#deleteToken, []),
+            Invocation.method(#logout, []),
             returnValue: _i3.Future<void>.value(),
             returnValueForMissingStub: _i3.Future<void>.value(),
           )
