diff --git a/src/models/data/crafted-items.d.ts b/src/models/data/crafted-items.d.ts
index f46084e..e363f9d 100644
--- a/src/models/data/crafted-items.d.ts
+++ b/src/models/data/crafted-items.d.ts
@@ -1,5 +1,4 @@
 // Type definitions for crafted-items.js
 import { CraftedItemCard } from '@/models/types/cards';
 
-declare const craftedItems: CraftedItemCard[];
-export default craftedItems;
+export const craftedItems: CraftedItemCard[];
