diff --git a/.cursor/rules/common-errors.mdc b/.cursor/rules/common-errors.mdc
new file mode 100644
index 0000000..4a72290
--- /dev/null
+++ b/.cursor/rules/common-errors.mdc
@@ -0,0 +1,7 @@
+---
+description: Rules describing how to respond to common errors
+alwaysApply: false
+---
+
+
+* If you see the error `ENFILE: file table overflow, watch warning` stop the dev server, run `npx nuxi cleanup`, and start the server again. This is especially common after writing E2E tests.
\ No newline at end of file
