diff --git a/.cursor/rules/database.mdc b/.cursor/rules/database.mdc
index f44ce65..7850774 100644
--- a/.cursor/rules/database.mdc
+++ b/.cursor/rules/database.mdc
@@ -5,6 +5,7 @@ alwaysApply: false
 ---
 * This project uses drizzle as a databse ORM. 
 * Project Schema - [drizzle-schema.ts](mdc:server/db/drizzle-schema.ts)
+* Changes to the schema should be made in the schema file and then run `npx drizzle-kit push` to update the database.
 * On the server side, call the auto-imported `useDb()` function
 * Seeds should be created as nitro tasks:
 
@@ -21,4 +22,10 @@ export default defineTask({
     return { result: "Success" };
   },
 });
-```
\ No newline at end of file
+```
+
+* Seeds can be run by making a request to the `/_nitro/tasks/[seed-name]` endpoint. For example:
+
+```bash
+curl -X GET http://localhost:3000/_nitro/tasks/seed:posts
+```
