diff --git a/lib/src/screens/loading_screen/loading_route.dart b/lib/src/screens/loading_screen/loading_route.dart
new file mode 100644
index 0000000..9e4040c
--- /dev/null
+++ b/lib/src/screens/loading_screen/loading_route.dart
@@ -0,0 +1,7 @@
+import 'package:gh3/src/routing/app_route.dart';
+
+/// Typed route for the loading screen
+class LoadingRoute extends AppRoute {
+  @override
+  String get path => '/loading';
+}
