diff --git a/test/screens/home_screen/home_route_test.dart b/test/screens/home_screen/home_route_test.dart
new file mode 100644
index 0000000..7f6ee2f
--- /dev/null
+++ b/test/screens/home_screen/home_route_test.dart
@@ -0,0 +1,11 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:gh3/src/screens/home_screen/home_route.dart';
+
+void main() {
+  group('HomeRoute', () {
+    test('should return correct path', () {
+      final homeRoute = HomeRoute();
+      expect(homeRoute.path, equals('/'));
+    });
+  });
+}
