diff --git a/test/screens/loading_screen/loading_route_test.dart b/test/screens/loading_screen/loading_route_test.dart
new file mode 100644
index 0000000..e7e62e5
--- /dev/null
+++ b/test/screens/loading_screen/loading_route_test.dart
@@ -0,0 +1,11 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:gh3/src/screens/loading_screen/loading_route.dart';
+
+void main() {
+  group('LoadingRoute', () {
+    test('should return correct path', () {
+      final loadingRoute = LoadingRoute();
+      expect(loadingRoute.path, equals('/loading'));
+    });
+  });
+}
