diff --git a/SwiftFlash/SwiftFlash/ToolbarConfigurationService.swift b/SwiftFlash/SwiftFlash/ToolbarConfigurationService.swift
index 1abbc5f..2e33192 100644
--- a/SwiftFlash/SwiftFlash/ToolbarConfigurationService.swift
+++ b/SwiftFlash/SwiftFlash/ToolbarConfigurationService.swift
@@ -9,10 +9,9 @@ class ToolbarConfigurationService {
     // Default toolbar configuration
     var toolbarItems: [String] = [
         "refresh",
-        "space1", 
+        "flexibleSpace",
         "flash",
         "eject",
-        "flexibleSpace",
         "tags",
         "debug",
         "about",
@@ -36,10 +35,9 @@ class ToolbarConfigurationService {
     func resetToDefault() {
         toolbarItems = [
             "refresh",
-            "space1", 
+            "flexibleSpace",
             "flash",
             "eject",
-            "flexibleSpace",
             "tags",
             "debug",
             "about",
