diff --git a/src/app.module.ts b/src/app.module.ts
index f1fee05..70e75af 100644
--- a/src/app.module.ts
+++ b/src/app.module.ts
@@ -5,6 +5,7 @@ import { TerminusModule } from '@nestjs/terminus';
 import { HealthCommand } from './commands/health.command';
 import { AuthModule } from './modules/auth/auth.module';
 import { InfoModule } from './modules/info/info.module';
+import { BuildModule } from './modules/build/build.module';
 import { ContextModule } from './modules/context/context.module';
 import { DeployModule } from './modules/deploy/deploy.module';
 
@@ -19,6 +20,7 @@ import { DeployModule } from './modules/deploy/deploy.module';
     InfoModule,
     ContextModule,
     DeployModule,
+    BuildModule,
   ],
   providers: [HealthCommand],
 })
