diff --git a/src/modules/build/build.module.ts b/src/modules/build/build.module.ts
new file mode 100644
index 0000000..2f10ce4
--- /dev/null
+++ b/src/modules/build/build.module.ts
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
+
+import { BuildCommand } from './commands/build.command';
+
+@Module({
+  providers: [BuildCommand],
+  exports: [BuildCommand],
+})
+export class BuildModule {}
\ No newline at end of file
