diff --git a/.cursor/rules/components.mdc b/.cursor/rules/components.mdc
index b93c988..0dd2485 100644
--- a/.cursor/rules/components.mdc
+++ b/.cursor/rules/components.mdc
@@ -1,5 +1,6 @@
 ---
-description:
-globs:
+description: rules for working with components 
+globs: 
 alwaysApply: false
 ---
+* When creating a brand new stand alone component (that's not a child/tightly coupled component), use the `nlx hygen component new --name [ComponentNameHere]` from package.json
\ No newline at end of file
