diff --git a/.cursor/rules/api.mdc b/.cursor/rules/api.mdc
index 797e37b..0c1b10d 100644
--- a/.cursor/rules/api.mdc
+++ b/.cursor/rules/api.mdc
@@ -6,6 +6,7 @@ alwaysApply: false
 * All API endpoints follow the OpenAPI specification.
 * When creating new resources (like users, posts, etc) use the following command to bootsrap all CRUD API endpoints: `npx hygen resource new --name [singular-resource-name-here]`
 * When creating one-off API endpoints (not full resources) bootstrap the API file with the command: `npx hygen api new --path [path-relative to server/api] --method [get, post, put, delete]`
+* ALWAYS read the created files after running the hygen command.
 * If you need to create an API endpoint that supports streaming:
   *  Don't use one of the hygen commands
   *  DO use the Nuxt MCP for reference
