diff --git a/go.mod b/go.mod
index 01d0a2a..9d4af84 100644
--- a/go.mod
+++ b/go.mod
@@ -14,6 +14,13 @@ require (
 	golang.org/x/crypto v0.40.0
 )
 
+require (
+	github.com/go-gorp/gorp/v3 v3.1.0 // indirect
+	github.com/inconshreveable/mousetrap v1.1.0 // indirect
+	github.com/rubenv/sql-migrate v1.8.0 // indirect
+	github.com/spf13/cobra v1.9.1 // indirect
+)
+
 require (
 	github.com/KyleBanks/depth v1.2.1 // indirect
 	github.com/PuerkitoBio/purell v1.1.1 // indirect
@@ -33,6 +40,7 @@ require (
 	github.com/go-playground/validator/v10 v10.27.0 // indirect
 	github.com/go-viper/mapstructure/v2 v2.2.1 // indirect
 	github.com/golang/mock v1.6.0 // indirect
+	github.com/jmoiron/sqlx v1.4.0
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/labstack/gommon v0.4.2 // indirect
 	github.com/leodido/go-urn v1.4.0 // indirect
