diff --git a/migration/.cursor/rules/miration.mdc b/migration/.cursor/rules/miration.mdc
new file mode 100644
index 0000000..fa6e5d6
--- /dev/null
+++ b/migration/.cursor/rules/miration.mdc
@@ -0,0 +1,19 @@
+# Migration
+
+This rules represents how database migration applied. Included : 
+- What library used
+- How migration file generated
+- How miggration applied (up and down)
+
+
+## Libraries 
+
+The migration process utilize https://github.com/rubenv/sql-migrate library. All migration file stored in ``migration/`` directory
+
+## How to generate migration file 
+
+```bash
+$ sql-migrate new <migration_name>
+```
+
+## How Migration Applied
\ No newline at end of file
