commit 3e4c749d1cab16a02a8efd6509049fa29b388230
Author:     Filip Gibki <filip@solutions4delivery.com>
AuthorDate: Fri Feb 28 11:31:14 2025 +0100
Commit:     Filip Gibki <filip@solutions4delivery.com>
CommitDate: Fri Feb 28 11:31:33 2025 +0100

    Updated Cursor rules

diff --git a/.cursor/rules/general.mdc b/.cursor/rules/general.mdc
new file mode 100644
index 0000000..f7ab6ee
--- /dev/null
+++ b/.cursor/rules/general.mdc
@@ -0,0 +1,8 @@
+---
+description: Everything
+globs: 
+alwaysApply: true
+---
+# CLI
+- We are on Windows so use power shell
+- Do not try to run web server, let user do it for you


commit ca298cb56ecdba7b6557acff0bda660303d33826
Author:     Filip Gibki <filip@solutions4delivery.com>
AuthorDate: Sat Mar 1 21:58:13 2025 +0100
Commit:     Filip Gibki <filip@solutions4delivery.com>
CommitDate: Sat Mar 1 21:58:13 2025 +0100

    Updated cursor rules

diff --git a/.cursor/rules/general.mdc b/.cursor/rules/general.mdc
index f7ab6ee..5f7c0dd 100644
--- a/.cursor/rules/general.mdc
+++ b/.cursor/rules/general.mdc
@@ -1,8 +1,14 @@
 ---
-description: Everything
+description: General rules for every request
 globs: 
 alwaysApply: true
 ---
+# Reference architecture
+1. Check [architecture.md](mdc:docs/architecture.md) for reference to make sure you are making changes in the right places.
+
 # CLI
-- We are on Windows so use power shell
-- Do not try to run web server, let user do it for you
+1. We are on Windows so use powershell commands.
+
+# End-to-end testing
+1. Use following command to start web server before executing tests `$env:FLASK_ENV="development"; python -m src.app`
+


commit b0832e264b10297aa1a04eeff3297b34c9a7c719
Author:     Filip Gibki <filip@solutions4delivery.com>
AuthorDate: Wed Mar 5 21:13:07 2025 +0100
Commit:     Filip Gibki <filip@solutions4delivery.com>
CommitDate: Wed Mar 5 21:13:07 2025 +0100

    docs: apply consistent commenting standard across Python codebase

diff --git a/.cursor/rules/general.mdc b/.cursor/rules/general.mdc
index 5f7c0dd..2e7e764 100644
--- a/.cursor/rules/general.mdc
+++ b/.cursor/rules/general.mdc
@@ -4,7 +4,7 @@ globs:
 alwaysApply: true
 ---
 # Reference architecture
-1. Check [architecture.md](mdc:docs/architecture.md) for reference to make sure you are making changes in the right places.
+1. Check [architecture.md](mdc:docs/architecture.md) for reference about project structure, standards and used technologies.
 
 # CLI
 1. We are on Windows so use powershell commands.


