diff --git a/src/main/utils/config-manager.ts b/src/main/utils/config-manager.ts
index 800f1af..213684d 100644
--- a/src/main/utils/config-manager.ts
+++ b/src/main/utils/config-manager.ts
@@ -12,7 +12,8 @@ export class ConfigManager {
     this.configFilePath = path.join(userDataPath, 'app-config.json');
     this.defaultConfig = {
       autoNumberDuplicateFiles: true,
-      defaultSaveLocation: app.getPath('downloads')
+      defaultSaveLocation: app.getPath('downloads'),
+      theme: 'system'
     };
   }
 
