diff --git a/lib/src/init.config.dart b/lib/src/init.config.dart
index c3d62a9..48ec81e 100644
--- a/lib/src/init.config.dart
+++ b/lib/src/init.config.dart
@@ -12,6 +12,7 @@
 import 'package:ferry/ferry.dart' as _i25;
 import 'package:get_it/get_it.dart' as _i174;
 import 'package:gh3/src/env/env.dart' as _i589;
+import 'package:gh3/src/routing/route_collection_service.dart' as _i482;
 import 'package:gh3/src/screens/home_screen/home_viewmodel.dart' as _i630;
 import 'package:gh3/src/services/auth_service.dart' as _i336;
 import 'package:gh3/src/services/ferry_client_service.dart' as _i564;
@@ -33,11 +34,12 @@ extension GetItInjectableX on _i174.GetIt {
     final envModule = _$EnvModule();
     final githubAuthHttpClientModule = _$GithubAuthHttpClientModule();
     final ferryModule = _$FerryModule();
+    gh.factory<_i482.RouteCollectionService>(
+      () => _i482.RouteCollectionService(),
+    );
     gh.lazySingleton<_i589.Env>(() => envModule.env);
     gh.lazySingleton<_i519.Client>(() => githubAuthHttpClientModule.httpClient);
-    gh.lazySingleton<_i1066.DefaultTimerService>(
-      () => _i1066.DefaultTimerService(),
-    );
+    gh.lazySingleton<_i1066.TimerService>(() => _i1066.DefaultTimerService());
     gh.lazySingleton<_i895.ITokenStorage>(() => _i895.PrefsTokenStorage());
     gh.factory<String>(
       () => envModule.githubClientId,
