diff --git a/lib/src/services/timer_service.dart b/lib/src/services/timer_service.dart
index 7b06675..e19a812 100644
--- a/lib/src/services/timer_service.dart
+++ b/lib/src/services/timer_service.dart
@@ -12,7 +12,7 @@ abstract class TimerService {
 }
 
 /// Default implementation of TimerService using dart:async.
-@lazySingleton
+@LazySingleton(as: TimerService)
 class DefaultTimerService implements TimerService {
   @override
   Future<void> delay(Duration duration) async {
