diff --git a/lib/src/init.config.dart b/lib/src/init.config.dart
index 4141681..8c55607 100644
--- a/lib/src/init.config.dart
+++ b/lib/src/init.config.dart
@@ -79,16 +79,16 @@ extension GetItInjectableX on _i174.GetIt {
         gh<_i1066.TimerService>(),
       ),
     );
-    gh.factory<_i850.AuthViewModel>(
-      () => _i850.AuthViewModel(gh<_i336.AuthService>()),
-    );
     gh.factory<_i1013.HomeViewModelFactory>(
       () => _i1013.HomeViewModelFactory(gh<_i336.AuthService>()),
     );
     gh.factory<_i564.FerryClientService>(
       () => _i564.FerryClientService(gh<_i336.AuthService>()),
     );
-    gh.factory<_i492.LoadingRouteProvider>(
+    gh.lazySingleton<_i850.AuthViewModel>(
+      () => _i850.AuthViewModel(gh<_i336.AuthService>()),
+    );
+    gh.singleton<_i518.RouteProvider>(
       () => _i492.LoadingRouteProvider(gh<_i850.AuthViewModel>()),
       instanceName: 'LoadingRouteProvider',
     );
@@ -98,7 +98,7 @@ extension GetItInjectableX on _i174.GetIt {
     gh.factory<_i627.UserDetailsViewModelFactory>(
       () => _i627.UserDetailsViewModelFactory(gh<_i25.Client>()),
     );
-    gh.factory<_i518.RouteProvider>(
+    gh.singleton<_i518.RouteProvider>(
       () => _i176.HomeRouteProvider(
         gh<_i1013.HomeViewModelFactory>(),
         gh<_i850.AuthViewModel>(),
@@ -112,13 +112,13 @@ extension GetItInjectableX on _i174.GetIt {
         gh<_i850.AuthViewModel>(),
       ),
     );
-    gh.factory<_i518.RouteProvider>(
+    gh.singleton<_i518.RouteProvider>(
       () => _i57.UserDetailsRouteProvider(
         gh<_i627.UserDetailsViewModelFactory>(),
       ),
       instanceName: 'UserDetailsRouteProvider',
     );
-    gh.factory<_i518.RouteProvider>(
+    gh.singleton<_i518.RouteProvider>(
       () => _i468.LoginRouteProvider(gh<_i76.LoginViewModelFactory>()),
       instanceName: 'LoginRouteProvider',
     );
