diff --git a/lib/src/screens/home_screen/home_route_provider.dart b/lib/src/screens/home_screen/home_route_provider.dart
index 11d5356..ca5739b 100644
--- a/lib/src/screens/home_screen/home_route_provider.dart
+++ b/lib/src/screens/home_screen/home_route_provider.dart
@@ -8,7 +8,7 @@ import 'home_screen.dart';
 /// Route provider for the home screen module.
 /// Implements RouteProvider interface to provide modular route configuration.
 @Named("HomeRouteProvider")
-@Injectable(as: RouteProvider)
+@Singleton(as: RouteProvider)
 class HomeRouteProvider implements RouteProvider {
   final HomeViewModelFactory _homeViewModelFactory;
   final AuthViewModel _authViewModel;
