diff --git a/lib/src/screens/login_screen/login_route_provider.dart b/lib/src/screens/login_screen/login_route_provider.dart
index 6ad82f3..43a4bdd 100644
--- a/lib/src/screens/login_screen/login_route_provider.dart
+++ b/lib/src/screens/login_screen/login_route_provider.dart
@@ -7,7 +7,7 @@ import 'login_screen.dart';
 /// Route provider for the login screen module.
 /// Implements RouteProvider interface to provide modular route configuration.
 @Named("LoginRouteProvider")
-@Injectable(as: RouteProvider)
+@Singleton(as: RouteProvider)
 class LoginRouteProvider implements RouteProvider {
   final LoginViewModelFactory _loginViewModelFactory;
 
