diff --git a/lib/src/screens/user_details/user_details_route_provider.dart b/lib/src/screens/user_details/user_details_route_provider.dart
index 5efcfff..b4df532 100644
--- a/lib/src/screens/user_details/user_details_route_provider.dart
+++ b/lib/src/screens/user_details/user_details_route_provider.dart
@@ -8,7 +8,7 @@ import 'user_details_screen.dart';
 /// Implements RouteProvider interface to provide modular route configuration
 /// with parameterized routes.
 @Named("UserDetailsRouteProvider")
-@Injectable(as: RouteProvider)
+@Singleton(as: RouteProvider)
 class UserDetailsRouteProvider implements RouteProvider {
   final UserDetailsViewModelFactory _userDetailsViewModelFactory;
 
