diff --git a/game/tictactoe_test.go b/game/tictactoe_test.go
index a75e517..e5590c1 100644
--- a/game/tictactoe_test.go
+++ b/game/tictactoe_test.go
@@ -1,8 +1,9 @@
 package game
 
 import (
-	"github.com/stretchr/testify/assert"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestText(t *testing.T) {
@@ -16,3 +17,11 @@ func TestInitialBoard(t *testing.T) {
 
 	assert.Equal(t, expected, game.Board(), "should start with empty board")
 }
+
+func TestMakeMove(t *testing.T) {
+	game := NewGame()
+	game.MakeMove(4) // center cell
+
+	board := game.Board()
+	assert.Equal(t, "X", board[4], "should place X in the center cell")
+}
