diff --git a/cspell.config.js b/cspell.config.js
new file mode 100644
index 0000000..c38508b
--- /dev/null
+++ b/cspell.config.js
@@ -0,0 +1,14 @@
+export default {
+	dictionaries: ['project'],
+	dictionaryDefinitions: [
+		{
+			addWords: true,
+			name: 'project',
+			path: './dictionary.txt',
+		},
+	],
+	ignorePaths: ['**/dist/', '**/src/data/', '**/src/locales/'],
+	import: ['@somehow-digital/cspell-dictionary'],
+	language: 'en',
+	version: '0.2',
+};
