diff --git a/src/router/index.ts b/src/router/index.ts
new file mode 100644
index 0000000..7310ae9
--- /dev/null
+++ b/src/router/index.ts
@@ -0,0 +1,16 @@
+import { createRouter, createWebHistory } from 'vue-router';
+
+import HomeView from '../views/home-view.vue';
+
+const router = createRouter({
+	history: createWebHistory(import.meta.env.BASE_URL),
+	routes: [
+		{
+			component: HomeView,
+			name: 'home',
+			path: '/',
+		},
+	],
+});
+
+export default router;
