diff --git a/vite.config.ts b/vite.config.ts
new file mode 100644
index 0000000..907fbdc
--- /dev/null
+++ b/vite.config.ts
@@ -0,0 +1,14 @@
+import vue from '@vitejs/plugin-vue';
+import { fileURLToPath, URL } from 'node:url';
+import unocss from 'unocss/vite';
+import { defineConfig } from 'vite';
+import tools from 'vite-plugin-vue-devtools';
+
+export default defineConfig({
+	plugins: [vue(), tools(), unocss()],
+	resolve: {
+		alias: {
+			'@': fileURLToPath(new URL('src', import.meta.url)),
+		},
+	},
+});
