diff --git a/lib/src/init.config.dart b/lib/src/init.config.dart
index 94e14ad..8ca552c 100644
--- a/lib/src/init.config.dart
+++ b/lib/src/init.config.dart
@@ -88,12 +88,12 @@ extension GetItInjectableX on _i174.GetIt {
       () async =>
           _i627.UserDetailsViewModelFactory(await getAsync<_i25.Client>()),
     );
-    gh.factoryAsync<_i1013.HomeViewModelFactory>(
-      () async => _i1013.HomeViewModelFactory(await getAsync<_i25.Client>()),
-    );
     gh.factory<_i850.AuthViewModel>(
       () => _i850.AuthViewModel(gh<_i336.AuthService>()),
     );
+    gh.factory<_i1013.HomeViewModelFactory>(
+      () => _i1013.HomeViewModelFactory(gh<_i336.AuthService>()),
+    );
     gh.factoryAsync<_i57.UserDetailsRouteProvider>(
       () async => _i57.UserDetailsRouteProvider(
         await getAsync<_i627.UserDetailsViewModelFactory>(),
@@ -106,9 +106,9 @@ extension GetItInjectableX on _i174.GetIt {
         gh<_i850.AuthViewModel>(),
       ),
     );
-    gh.factoryAsync<_i176.HomeRouteProvider>(
-      () async => _i176.HomeRouteProvider(
-        await getAsync<_i1013.HomeViewModelFactory>(),
+    gh.factory<_i176.HomeRouteProvider>(
+      () => _i176.HomeRouteProvider(
+        gh<_i1013.HomeViewModelFactory>(),
         gh<_i850.AuthViewModel>(),
       ),
     );
