diff --git a/lib/src/screens/home_screen/home_viewmodel_factory.dart b/lib/src/screens/home_screen/home_viewmodel_factory.dart
index 4856580..e152d95 100644
--- a/lib/src/screens/home_screen/home_viewmodel_factory.dart
+++ b/lib/src/screens/home_screen/home_viewmodel_factory.dart
@@ -1,14 +1,14 @@
-import 'package:ferry/ferry.dart';
 import 'package:injectable/injectable.dart';
 import 'home_viewmodel.dart';
+import '../../services/auth_service.dart';
 
 @injectable
 class HomeViewModelFactory {
-  final Client _ferryClient;
+  final AuthService _authService;
 
-  HomeViewModelFactory(this._ferryClient);
+  HomeViewModelFactory(this._authService);
 
   HomeViewModel create() {
-    return HomeViewModel(_ferryClient);
+    return HomeViewModel(_authService);
   }
 }
