diff --git a/lib/src/screens/home_screen/widgets/section_header.dart b/lib/src/screens/home_screen/widgets/section_header.dart
new file mode 100644
index 0000000..f9438cc
--- /dev/null
+++ b/lib/src/screens/home_screen/widgets/section_header.dart
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+class SectionHeader extends StatelessWidget {
+  final String title;
+
+  const SectionHeader({super.key, required this.title});
+
+  @override
+  Widget build(BuildContext context) {
+    return Text(
+      title,
+      style: Theme.of(
+        context,
+      ).textTheme.titleLarge?.copyWith(fontWeight: FontWeight.bold),
+    );
+  }
+}
