diff --git a/lib/src/screens/user_details/user_details_viewmodel.dart b/lib/src/screens/user_details/user_details_viewmodel.dart
index e03dfaf..9a0c71f 100644
--- a/lib/src/screens/user_details/user_details_viewmodel.dart
+++ b/lib/src/screens/user_details/user_details_viewmodel.dart
@@ -57,8 +57,9 @@ class UserDetailsViewModel extends DisposableViewModel {
     if (_userResult != null) {
       final linkException = _userResult!.linkException;
       final graphqlErrors = _userResult!.graphqlErrors;
-      
-      if (linkException != null || (graphqlErrors != null && graphqlErrors.isNotEmpty)) {
+
+      if (linkException != null ||
+          (graphqlErrors != null && graphqlErrors.isNotEmpty)) {
         return GraphQLErrorHandler.getErrorMessage(_userResult!);
       }
     }
@@ -67,8 +68,9 @@ class UserDetailsViewModel extends DisposableViewModel {
     if (_repositoriesResult != null) {
       final linkException = _repositoriesResult!.linkException;
       final graphqlErrors = _repositoriesResult!.graphqlErrors;
-      
-      if (linkException != null || (graphqlErrors != null && graphqlErrors.isNotEmpty)) {
+
+      if (linkException != null ||
+          (graphqlErrors != null && graphqlErrors.isNotEmpty)) {
         return GraphQLErrorHandler.getErrorMessage(_repositoriesResult!);
       }
     }
