diff --git a/lib/src/services/graphql_errors.dart b/lib/src/services/graphql_errors.dart
index 4649101..50fe4a7 100644
--- a/lib/src/services/graphql_errors.dart
+++ b/lib/src/services/graphql_errors.dart
@@ -40,6 +40,5 @@ class GraphQLValidationError extends GraphQLError {
 }
 
 class GraphQLOfflineError extends GraphQLError {
-  const GraphQLOfflineError(super.message)
-    : super(code: 'OFFLINE_ERROR');
+  const GraphQLOfflineError(super.message) : super(code: 'OFFLINE_ERROR');
 }
