diff --git a/test/screens/home_screen/home_route_provider_test.mocks.dart b/test/screens/home_screen/home_route_provider_test.mocks.dart
index 074e5e2..2af67dd 100644
--- a/test/screens/home_screen/home_route_provider_test.mocks.dart
+++ b/test/screens/home_screen/home_route_provider_test.mocks.dart
@@ -7,16 +7,14 @@ import 'dart:async' as _i7;
 import 'dart:ui' as _i8;
 
 import 'package:flutter/foundation.dart' as _i4;
-import 'package:flutter/src/widgets/notification_listener.dart' as _i10;
+import 'package:flutter/src/widgets/notification_listener.dart' as _i9;
 import 'package:flutter/widgets.dart' as _i3;
 import 'package:gh3/src/screens/app/auth_viewmodel.dart' as _i6;
-import 'package:gh3/src/screens/home_screen/__generated__/home_viewmodel.data.gql.dart'
-    as _i9;
 import 'package:gh3/src/screens/home_screen/home_viewmodel.dart' as _i2;
 import 'package:gh3/src/screens/home_screen/home_viewmodel_factory.dart' as _i5;
-import 'package:go_router/src/state.dart' as _i11;
+import 'package:go_router/src/state.dart' as _i10;
 import 'package:mockito/mockito.dart' as _i1;
-import 'package:mockito/src/dummies.dart' as _i12;
+import 'package:mockito/src/dummies.dart' as _i11;
 
 // ignore_for_file: type=lint
 // ignore_for_file: avoid_redundant_argument_values
@@ -202,29 +200,6 @@ class MockHomeViewModel extends _i1.Mock implements _i2.HomeViewModel {
     _i1.throwOnMissingStub(this);
   }
 
-  @override
-  List<_i9.GGetFollowingData_viewer_following_nodes> get followingUsers =>
-      (super.noSuchMethod(
-            Invocation.getter(#followingUsers),
-            returnValue: <_i9.GGetFollowingData_viewer_following_nodes>[],
-          )
-          as List<_i9.GGetFollowingData_viewer_following_nodes>);
-
-  @override
-  bool get isLoading =>
-      (super.noSuchMethod(Invocation.getter(#isLoading), returnValue: false)
-          as bool);
-
-  @override
-  bool get hasMore =>
-      (super.noSuchMethod(Invocation.getter(#hasMore), returnValue: false)
-          as bool);
-
-  @override
-  bool get isEmpty =>
-      (super.noSuchMethod(Invocation.getter(#isEmpty), returnValue: false)
-          as bool);
-
   @override
   bool get disposed =>
       (super.noSuchMethod(Invocation.getter(#disposed), returnValue: false)
@@ -235,39 +210,6 @@ class MockHomeViewModel extends _i1.Mock implements _i2.HomeViewModel {
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
           as bool);
 
-  @override
-  _i7.Future<void> loadFollowingUsers() =>
-      (super.noSuchMethod(
-            Invocation.method(#loadFollowingUsers, []),
-            returnValue: _i7.Future<void>.value(),
-            returnValueForMissingStub: _i7.Future<void>.value(),
-          )
-          as _i7.Future<void>);
-
-  @override
-  _i7.Future<void> loadMoreFollowingUsers() =>
-      (super.noSuchMethod(
-            Invocation.method(#loadMoreFollowingUsers, []),
-            returnValue: _i7.Future<void>.value(),
-            returnValueForMissingStub: _i7.Future<void>.value(),
-          )
-          as _i7.Future<void>);
-
-  @override
-  _i7.Future<void> refreshFollowingUsers() =>
-      (super.noSuchMethod(
-            Invocation.method(#refreshFollowingUsers, []),
-            returnValue: _i7.Future<void>.value(),
-            returnValueForMissingStub: _i7.Future<void>.value(),
-          )
-          as _i7.Future<void>);
-
-  @override
-  void clearError() => super.noSuchMethod(
-    Invocation.method(#clearError, []),
-    returnValueForMissingStub: null,
-  );
-
   @override
   void onDispose() => super.noSuchMethod(
     Invocation.method(#onDispose, []),
@@ -364,7 +306,7 @@ class MockBuildContext extends _i1.Mock implements _i3.BuildContext {
   );
 
   @override
-  void dispatchNotification(_i10.Notification? notification) =>
+  void dispatchNotification(_i9.Notification? notification) =>
       super.noSuchMethod(
         Invocation.method(#dispatchNotification, [notification]),
         returnValueForMissingStub: null,
@@ -426,7 +368,7 @@ class MockBuildContext extends _i1.Mock implements _i3.BuildContext {
 ///
 /// See the documentation for Mockito's code generation for more information.
 // ignore: must_be_immutable
-class MockGoRouterState extends _i1.Mock implements _i11.GoRouterState {
+class MockGoRouterState extends _i1.Mock implements _i10.GoRouterState {
   MockGoRouterState() {
     _i1.throwOnMissingStub(this);
   }
@@ -443,7 +385,7 @@ class MockGoRouterState extends _i1.Mock implements _i11.GoRouterState {
   String get matchedLocation =>
       (super.noSuchMethod(
             Invocation.getter(#matchedLocation),
-            returnValue: _i12.dummyValue<String>(
+            returnValue: _i11.dummyValue<String>(
               this,
               Invocation.getter(#matchedLocation),
             ),
@@ -486,7 +428,7 @@ class MockGoRouterState extends _i1.Mock implements _i11.GoRouterState {
                 #fragment: fragment,
               },
             ),
-            returnValue: _i12.dummyValue<String>(
+            returnValue: _i11.dummyValue<String>(
               this,
               Invocation.method(
                 #namedLocation,
