diff --git a/.cursor/rules/wf-comp.mdc b/.cursor/rules/wf-comp.mdc
index 5db4b06..655059e 100644
--- a/.cursor/rules/wf-comp.mdc
+++ b/.cursor/rules/wf-comp.mdc
@@ -6,4 +6,5 @@ alwaysApply: false
 * Create simple props down, events up components that don't interact with any API endpoints.
 * Optimize them as if their consuming components WILL be getting data from API endpoints
 * Add a page in the `app/pages/playground` directory with hardcoded dummy data passed in to the props and reacting to events.
+* You can make curl requests and use the playwright mcp to check the results of the demo page
 * ALWAYS read the [components.mdc](mdc:.cursor/rules/components.mdc)
\ No newline at end of file
