diff --git a/.cursor/rules/database.mdc b/.cursor/rules/database.mdc
index 2646371..f44ce65 100644
--- a/.cursor/rules/database.mdc
+++ b/.cursor/rules/database.mdc
@@ -1,8 +1,24 @@
 ---
-description: 
+description: rules for working with the database
 globs: 
-alwaysApply: true
+alwaysApply: false
 ---
 * This project uses drizzle as a databse ORM. 
 * Project Schema - [drizzle-schema.ts](mdc:server/db/drizzle-schema.ts)
-* On the server side, call the auto-imported `useDb()` function
\ No newline at end of file
+* On the server side, call the auto-imported `useDb()` function
+* Seeds should be created as nitro tasks:
+
+```ts
+// server/tasks/seed/posts.ts
+export default defineTask({
+  meta: {
+      name: "seed:posts",
+      description: "Seed Posts",
+    },
+  run({ payload, context }) {
+    const db = useDb();
+    // do seeding
+    return { result: "Success" };
+  },
+});
+```
\ No newline at end of file
