diff --git a/lib/src/screens/app/gh3_app.dart b/lib/src/screens/app/gh3_app.dart
index d21194e..f7b625e 100644
--- a/lib/src/screens/app/gh3_app.dart
+++ b/lib/src/screens/app/gh3_app.dart
@@ -5,6 +5,7 @@ import 'package:gh3/src/screens/home_screen/home_viewmodel.dart';
 import 'package:gh3/src/screens/loading_screen/loading_screen.dart';
 import 'package:gh3/src/screens/login_screen/login_screen.dart';
 import 'package:gh3/src/screens/login_screen/login_viewmodel.dart';
+import 'package:gh3/src/screens/user_details/user_details_screen.dart';
 import 'package:gh3/src/services/auth_service.dart';
 import 'package:gh3/src/services/github_auth_client.dart';
 import 'package:go_router/go_router.dart';
@@ -52,6 +53,12 @@ class Gh3App extends StatelessWidget {
             homeViewModel: homeViewModel,
           ),
         ),
+        GoRoute(
+          path: '/:login',
+          builder: (context, state) => UserDetailsScreen(
+            login: state.pathParameters['login']!,
+          ),
+        ),
       ],
       redirect: (context, state) {
         if (authViewModel.loading) {
