diff --git a/lib/src/screens/user_details/user_details_screen.dart b/lib/src/screens/user_details/user_details_screen.dart
new file mode 100644
index 0000000..ba6f437
--- /dev/null
+++ b/lib/src/screens/user_details/user_details_screen.dart
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+class UserDetailsScreen extends StatelessWidget {
+  final String login;
+  const UserDetailsScreen({Key? key, required this.login}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        leading: BackButton(
+          onPressed: () {
+            if (Navigator.of(context).canPop()) {
+              Navigator.of(context).pop();
+            } else {
+              context.go('/');
+            }
+          },
+        ),
+        title: const Text('User Details'),
+      ),
+      body: Center(
+        child: Text(
+          'User: $login',
+          style: Theme.of(context).textTheme.headlineMedium,
+        ),
+      ),
+    );
+  }
+} 
\ No newline at end of file
