diff --git a/src/app/routes/__root.tsx b/src/app/routes/__root.tsx
new file mode 100644
index 0000000..32ca5a9
--- /dev/null
+++ b/src/app/routes/__root.tsx
@@ -0,0 +1,15 @@
+import { Outlet, createRootRoute } from '@tanstack/react-router';
+import { TanStackRouterDevtools } from '@tanstack/react-router-devtools';
+
+export const Route = createRootRoute({
+  component: RootComponent,
+});
+
+function RootComponent() {
+  return (
+    <>
+      <Outlet />
+      <TanStackRouterDevtools />
+    </>
+  );
+}
