diff --git a/vite.config.ts b/vite.config.ts
new file mode 100644
index 0000000..933899e
--- /dev/null
+++ b/vite.config.ts
@@ -0,0 +1,20 @@
+import tailwindcss from '@tailwindcss/vite';
+import { TanStackRouterVite } from '@tanstack/router-plugin/vite';
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
+import tsconfigPaths from 'vite-tsconfig-paths';
+
+// https://vite.dev/config/
+export default defineConfig({
+  plugins: [
+    TanStackRouterVite({
+      target: 'react',
+      autoCodeSplitting: true,
+      routesDirectory: './src/app/routes',
+      generatedRouteTree: './src/shared/utils/router/routeTree.gen.ts',
+    }),
+    react(),
+    tailwindcss(),
+    tsconfigPaths(),
+  ],
+});
