diff --git a/src/shared/utils/event-bus-register.utils.ts b/src/shared/utils/event-bus-register.utils.ts
index a706ad1..9a9f286 100644
--- a/src/shared/utils/event-bus-register.utils.ts
+++ b/src/shared/utils/event-bus-register.utils.ts
@@ -1,5 +1,5 @@
 import * as vscode from 'vscode';
-import { messageBus } from '../../services/messageBus';
+import { messageBus } from '../../services/message-bus.service';
 
 /**
  * Registers a webview with the central message bus, forwards messages, and handles disposal.
