diff --git a/src/ui-providers/health-dashboard/index.ts b/src/ui-providers/health-dashboard/index.ts
index cf3fff9..be5474f 100644
--- a/src/ui-providers/health-dashboard/index.ts
+++ b/src/ui-providers/health-dashboard/index.ts
@@ -1,7 +1,7 @@
 import * as vscode from 'vscode';
 import { showHealthDashboard } from './health-dashboard.panel';
 import { OPEN_DASHBOARD_COMMAND } from '../../shared/commands';
-import { messageBus } from '../../services/messageBus';
+import { messageBus } from '../../services/message-bus.service';
 import { Events } from '../../shared/events';
 
 
