diff --git a/src/components/SkillCard.vue b/src/components/SkillCard.vue
index 95afe32..3421676 100644
--- a/src/components/SkillCard.vue
+++ b/src/components/SkillCard.vue
@@ -33,6 +33,7 @@
         <ColoredText v-if="skill.coldDownTurns != 0 && skill.remainingUses != skill.maxUses && !previewMode" :text="`/named{重整} ${skill.remainingColdDownTurns}/${skill.coldDownTurns}`"></ColoredText>
         <ColoredText v-else-if="skill.coldDownTurns != 0" :text="`/named{重整} ${skill.coldDownTurns} 回合`"></ColoredText>
         <ColoredText v-else-if="skill.remainingUses != Infinity" :text="`/named{消耗}`"></ColoredText>
+        <ColoredText v-if="skill.slowStart" text="/named{慢热}"></ColoredText>
         <br />
         <strong v-if="skill.maxUses === Infinity && skill.coldDownTurns == 0">无限</strong>
         <span v-else-if="previewMode">(装填 {{ skill.maxUses }}/{{ skill.maxUses }})</span>
