diff --git a/lib/src/screens/app/gh3_app.dart b/lib/src/screens/app/gh3_app.dart
index f7b625e..3a203a0 100644
--- a/lib/src/screens/app/gh3_app.dart
+++ b/lib/src/screens/app/gh3_app.dart
@@ -55,9 +55,8 @@ class Gh3App extends StatelessWidget {
         ),
         GoRoute(
           path: '/:login',
-          builder: (context, state) => UserDetailsScreen(
-            login: state.pathParameters['login']!,
-          ),
+          builder: (context, state) =>
+              UserDetailsScreen(login: state.pathParameters['login']!),
         ),
       ],
       redirect: (context, state) {
