diff --git a/.windsurfrules b/.windsurfrules
index 18f196f..5176aa6 100644
--- a/.windsurfrules
+++ b/.windsurfrules
@@ -1,15 +1,16 @@
 mportant Restrictions
 All tests are made in the @tests directory
 No mocking tests
-do not run or build the application
+
 do not execute git commands
-Minimise starting and stopping the npm server
+
 do not create new files
 do not create new css styles
 work with the existing design
 use existing componets
 do not run tests
-minial descricptions of work performed back to the user
+report only minial descricptions of work performed back to the user
 maintain consistancy with models in src\models\data use using 
 Do not hard code game values. They are in src\models\data.
 minimize ruinning shell commands
+AI\Current Game Architecture.md
\ No newline at end of file
