diff --git a/.cursorrules b/.cursorrules
index 865fb73..089d018 100644
--- a/.cursorrules
+++ b/.cursorrules
@@ -11,7 +11,7 @@ Dependencies: testify
 Process:
 1. All development must follow TDD (Test-Driven Development) principles:
    - Start by writing a failing test for the desired functionality.
-   - Implement the minimal code required to make the code compile.
+   - Implement the minimal code required to make the code compile BUT NOTHING MORE.
    - Watch the test fail.
    - Implement the minimal code required to make the test pass.
    - Watch the test pass.
