diff --git a/src/app/equations/equations_tests.py b/src/app/equations/equations_tests.py
index 2687fed..fd16324 100644
--- a/src/app/equations/equations_tests.py
+++ b/src/app/equations/equations_tests.py
@@ -1,4 +1,4 @@
-from src.app.equations.equations import generator, MathEquationGenerator
+from src.app.equations.equations_generator import generator, MathEquationGenerator
 
 configurations_advanced = [
     {
