diff --git a/lib/src/screens/home_screen/home_route_provider.dart b/lib/src/screens/home_screen/home_route_provider.dart
index 0b66fd7..11d5356 100644
--- a/lib/src/screens/home_screen/home_route_provider.dart
+++ b/lib/src/screens/home_screen/home_route_provider.dart
@@ -7,7 +7,8 @@ import 'home_screen.dart';
 
 /// Route provider for the home screen module.
 /// Implements RouteProvider interface to provide modular route configuration.
-@injectable
+@Named("HomeRouteProvider")
+@Injectable(as: RouteProvider)
 class HomeRouteProvider implements RouteProvider {
   final HomeViewModelFactory _homeViewModelFactory;
   final AuthViewModel _authViewModel;
