diff --git a/lib/src/screens/loading_screen/loading_route_provider.dart b/lib/src/screens/loading_screen/loading_route_provider.dart
index 75718ba..eee5e73 100644
--- a/lib/src/screens/loading_screen/loading_route_provider.dart
+++ b/lib/src/screens/loading_screen/loading_route_provider.dart
@@ -7,6 +7,8 @@ import 'loading_screen.dart';
 /// Route provider for the loading screen module.
 /// Implements RouteProvider interface to provide modular route configuration.
 @injectable
+@Named("LoadingRouteProvider")
+@Injectable(as: RouteProvider)
 class LoadingRouteProvider implements RouteProvider {
   final AuthViewModel _authViewModel;
 
