diff --git a/infrastructure/lib/grace-orchestration-stack.ts b/infrastructure/lib/grace-orchestration-stack.ts
index 688e194..4f884f0 100644
--- a/infrastructure/lib/grace-orchestration-stack.ts
+++ b/infrastructure/lib/grace-orchestration-stack.ts
@@ -58,7 +58,11 @@ export class GraceOrchestrationStack extends cdk.NestedStack {
       eventPattern: {
         source: ['aws.s3'],
         detailType: ['Object Created'],
-        resources: [props.dataBucket.bucketArn]
+        detail: {
+          bucket: {
+            name: [props.dataBucket.bucketName]
+          }
+        }
       },
       targets: [new targets.SfnStateMachine(this.stateMachine)]
     });
