diff --git a/.cursor/rules/config.mdc b/.cursor/rules/config.mdc
new file mode 100644
index 0000000..0a0f545
--- /dev/null
+++ b/.cursor/rules/config.mdc
@@ -0,0 +1,13 @@
+---
+description:
+globs:
+alwaysApply: false
+---
+
+## Testing Configuration
+* use .env.test for testing configuration
+* read .env.test.example for more information
+
+## App Configuration
+* use .env for app configuration
+* read .env.example for more information
