diff --git a/src/modules/build/build.module.ts b/src/modules/build/build.module.ts
index 2f10ce4..cf6b400 100644
--- a/src/modules/build/build.module.ts
+++ b/src/modules/build/build.module.ts
@@ -1,9 +1,10 @@
 import { Module } from '@nestjs/common';
 
 import { BuildCommand } from './commands/build.command';
+import { InteractiveService } from './services/interactive.service';
 
 @Module({
-  providers: [BuildCommand],
-  exports: [BuildCommand],
+  providers: [BuildCommand, InteractiveService],
+  exports: [BuildCommand, InteractiveService],
 })
 export class BuildModule {}
\ No newline at end of file
