diff --git a/vitest.e2e.config.ts b/vitest.e2e.config.ts
index c9c7bcd..92ea40d 100644
--- a/vitest.e2e.config.ts
+++ b/vitest.e2e.config.ts
@@ -3,8 +3,8 @@ import { defineConfig } from 'vitest/config';
 export default defineConfig({
   test: {
     include: ['**/*.e2e.spec.ts'],
-    testTimeout: 30000,
-    hookTimeout: 30000,
+    testTimeout: 30_000,
+    hookTimeout: 30_000,
     globals: true,
     environment: 'node',
   },
