diff --git a/.cursor/rules/general.mdc b/.cursor/rules/general.mdc
index 468feda..bd959c7 100644
--- a/.cursor/rules/general.mdc
+++ b/.cursor/rules/general.mdc
@@ -42,8 +42,16 @@ nuxt.config.ts
 
 * The dev server is already up and running on localhost:3000
 
+
+
 * Read and use the following cursor rules when needed based on description. It's VERY IMPORTANT that you think about if you need any of these and include them when related to the task at hand:
   * [api.mdc](mdc:.cursor/rules/api.mdc) - rules for working with API endpoitns
   * [testing.mdc](mdc:.cursor/rules/testing.mdc) - rules for writing unit and e2e tests
   * [database.mdc](mdc:.cursor/rules/database.mdc) - rules for interacting with the database
   * [components.mdc](mdc:.cursor/rules/components.mdc) - rules for working with components 
+
+
+## State Management
+
+* Only use state management external to components when necessary
+* Utilize caching with useFetch and useAsync data for as much state management for fetched data as possible otherwise use a custom composable and useState
