diff --git a/src/components/StartScreen.vue b/src/components/StartScreen.vue
index cdb29ee..a0ffea4 100644
--- a/src/components/StartScreen.vue
+++ b/src/components/StartScreen.vue
@@ -33,6 +33,8 @@
 
 import frontendEventBus from '../frontendEventBus.js';
 import ChangeLog from './ChangeLog.vue';
+import {backendGameState} from "../data/gameState";
+import backendEventBus from "../backendEventBus";
 
 export default {
   name: 'StartScreen',
@@ -128,13 +130,13 @@ export default {
           ],
           interval: 3500,
           onEnd: ()=>{
-            frontendEventBus.emit('start-game');
+            backendEventBus.emit('start-game');
             this.isGameStarting = false;
           }
         })}, 4000);
       } else {
         // 不搞些花里胡哨的，直接开始
-        frontendEventBus.emit('start-game');
+        backendEventBus.emit('start-game');
       }
     },
     
