diff --git a/src/main.js b/src/main.js
index 4e920e8..2832be4 100644
--- a/src/main.js
+++ b/src/main.js
@@ -9,6 +9,7 @@ import './assets/main.css'
 import './assets/common.css'
 import SkillManager from './data/skillManager.js'
 import { initAnimationDispatcher } from './data/animationDispatcher.js'
+import {initGameFlowListeners} from "./game";
 
 // 创建路由
 const routes = [
@@ -39,4 +40,7 @@ SkillManager.loadAllSkills().then(skillManager => {
   
   // 挂载应用
   app.mount('#app');
-});
\ No newline at end of file
+});
+
+// 初始化后端游戏流程在backendEventBus上的监听器
+initGameFlowListeners();
\ No newline at end of file
