diff --git a/.cursor/rules/testing.mdc b/.cursor/rules/testing.mdc
new file mode 100644
index 0000000..21a0148
--- /dev/null
+++ b/.cursor/rules/testing.mdc
@@ -0,0 +1,7 @@
+---
+description: 
+globs: 
+alwaysApply: false
+---
+* If ever in doubt, use context7 mcp or search the web to help write a unit test
+* More tests documentation found in [README.md](mdc:tests/README.md).(tests/README.md)
\ No newline at end of file
