diff --git a/app/layouts/default.vue b/app/layouts/default.vue
index 1a50727..75f7606 100644
--- a/app/layouts/default.vue
+++ b/app/layouts/default.vue
@@ -4,16 +4,6 @@
       <div class="flex items-center space-x-4">
         <h2 class="text-lg font-semibold">My Awesome Project</h2>
       </div>
-      <div class="ml-auto flex items-center space-x-4">
-        <Button variant="ghost" size="sm">
-          <Link class="h-4 w-4" />
-          <span class="sr-only">GitHub</span>
-        </Button>
-        <Button variant="ghost" size="sm">
-          <User class="h-4 w-4" />
-          <span class="sr-only">Profile</span>
-        </Button>
-      </div>
     </div>
   </nav>
   <div>
