diff --git a/nuxt.config.ts b/nuxt.config.ts
index fa8e15c..ed9961b 100644
--- a/nuxt.config.ts
+++ b/nuxt.config.ts
@@ -11,7 +11,7 @@ export default defineNuxtConfig({
   vite: {
     plugins: [tailwindcss()],
   },
-  modules: ["@nuxt/eslint", "shadcn-nuxt"],
+  modules: ["@nuxt/eslint", "shadcn-nuxt", "@nuxt/test-utils/module"],
   shadcn: {
     /**
      * Prefix for all the imported component
