diff --git a/app/src/setupTests.js b/app/src/setupTests.js
new file mode 100644
index 0000000..8f2609b
--- /dev/null
+++ b/app/src/setupTests.js
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
