diff --git a/lib/src/init.config.dart b/lib/src/init.config.dart
index 5b682c1..cb550ae 100644
--- a/lib/src/init.config.dart
+++ b/lib/src/init.config.dart
@@ -47,12 +47,12 @@ extension GetItInjectableX on _i174.GetIt {
     final envModule = _$EnvModule();
     final githubAuthHttpClientModule = _$GithubAuthHttpClientModule();
     final ferryModule = _$FerryModule();
-    gh.factory<_i482.RouteCollectionService>(
-      () => _i482.RouteCollectionService(),
-    );
     gh.factory<_i627.UserDetailsViewModelFactory>(
       () => _i627.UserDetailsViewModelFactory(),
     );
+    gh.factory<_i482.RouteCollectionService>(
+      () => _i482.RouteCollectionService(),
+    );
     gh.lazySingleton<_i589.Env>(() => envModule.env);
     gh.lazySingleton<_i519.Client>(() => githubAuthHttpClientModule.httpClient);
     gh.lazySingleton<_i1066.TimerService>(() => _i1066.DefaultTimerService());
