diff --git a/test/screens/home_screen/home_viewmodel_test.mocks.dart b/test/screens/home_screen/home_viewmodel_test.mocks.dart
index 5f1d2ed..9be1fca 100644
--- a/test/screens/home_screen/home_viewmodel_test.mocks.dart
+++ b/test/screens/home_screen/home_viewmodel_test.mocks.dart
@@ -49,15 +49,15 @@ class _FakeTypedLink_3 extends _i1.SmartFake implements _i2.TypedLink {
 ///
 /// See the documentation for Mockito's code generation for more information.
 class MockClient extends _i1.Mock implements _i4.Client {
+  MockClient() {
+    _i1.throwOnMissingStub(this);
+  }
+
   @override
   _i2.Link get link =>
       (super.noSuchMethod(
             Invocation.getter(#link),
             returnValue: _FakeLink_0(this, Invocation.getter(#link)),
-            returnValueForMissingStub: _FakeLink_0(
-              this,
-              Invocation.getter(#link),
-            ),
           )
           as _i2.Link);
 
@@ -66,7 +66,6 @@ class MockClient extends _i1.Mock implements _i4.Client {
       (super.noSuchMethod(
             Invocation.getter(#typePolicies),
             returnValue: <String, _i2.TypePolicy>{},
-            returnValueForMissingStub: <String, _i2.TypePolicy>{},
           )
           as Map<String, _i2.TypePolicy>);
 
@@ -75,7 +74,6 @@ class MockClient extends _i1.Mock implements _i4.Client {
       (super.noSuchMethod(
             Invocation.getter(#updateCacheHandlers),
             returnValue: <String, Function>{},
-            returnValueForMissingStub: <String, Function>{},
           )
           as Map<String, Function>);
 
@@ -84,17 +82,12 @@ class MockClient extends _i1.Mock implements _i4.Client {
       (super.noSuchMethod(
             Invocation.getter(#defaultFetchPolicies),
             returnValue: <_i5.OperationType, _i2.FetchPolicy>{},
-            returnValueForMissingStub: <_i5.OperationType, _i2.FetchPolicy>{},
           )
           as Map<_i5.OperationType, _i2.FetchPolicy>);
 
   @override
   bool get addTypename =>
-      (super.noSuchMethod(
-            Invocation.getter(#addTypename),
-            returnValue: false,
-            returnValueForMissingStub: false,
-          )
+      (super.noSuchMethod(Invocation.getter(#addTypename), returnValue: false)
           as bool);
 
   @override
@@ -102,10 +95,6 @@ class MockClient extends _i1.Mock implements _i4.Client {
       (super.noSuchMethod(
             Invocation.getter(#cache),
             returnValue: _FakeCache_1(this, Invocation.getter(#cache)),
-            returnValueForMissingStub: _FakeCache_1(
-              this,
-              Invocation.getter(#cache),
-            ),
           )
           as _i2.Cache);
 
@@ -119,11 +108,6 @@ class MockClient extends _i1.Mock implements _i4.Client {
                   this,
                   Invocation.getter(#requestController),
                 ),
-            returnValueForMissingStub:
-                _FakeStreamController_2<_i2.OperationRequest<dynamic, dynamic>>(
-                  this,
-                  Invocation.getter(#requestController),
-                ),
           )
           as _i3.StreamController<_i2.OperationRequest<dynamic, dynamic>>);
 
@@ -160,8 +144,6 @@ class MockClient extends _i1.Mock implements _i4.Client {
             Invocation.method(#request, [request, forward]),
             returnValue:
                 _i3.Stream<_i2.OperationResponse<TData, TVars>>.empty(),
-            returnValueForMissingStub:
-                _i3.Stream<_i2.OperationResponse<TData, TVars>>.empty(),
           )
           as _i3.Stream<_i2.OperationResponse<TData, TVars>>);
 
@@ -173,10 +155,6 @@ class MockClient extends _i1.Mock implements _i4.Client {
               this,
               Invocation.method(#concat, [next]),
             ),
-            returnValueForMissingStub: _FakeTypedLink_3(
-              this,
-              Invocation.method(#concat, [next]),
-            ),
           )
           as _i2.TypedLink);
 
@@ -188,10 +166,6 @@ class MockClient extends _i1.Mock implements _i4.Client {
               this,
               Invocation.method(#route, [route]),
             ),
-            returnValueForMissingStub: _FakeTypedLink_3(
-              this,
-              Invocation.method(#route, [route]),
-            ),
           )
           as _i2.TypedLink);
 
@@ -207,10 +181,6 @@ class MockClient extends _i1.Mock implements _i4.Client {
               this,
               Invocation.method(#split, [test, left, right]),
             ),
-            returnValueForMissingStub: _FakeTypedLink_3(
-              this,
-              Invocation.method(#split, [test, left, right]),
-            ),
           )
           as _i2.TypedLink);
 }
