diff --git a/test/screens/login_screen/login_route_provider_test.mocks.dart b/test/screens/login_screen/login_route_provider_test.mocks.dart
index e072a49..13a30af 100644
--- a/test/screens/login_screen/login_route_provider_test.mocks.dart
+++ b/test/screens/login_screen/login_route_provider_test.mocks.dart
@@ -80,12 +80,6 @@ class MockLoginViewModel extends _i1.Mock implements _i2.LoginViewModel {
       (super.noSuchMethod(Invocation.getter(#hasListeners), returnValue: false)
           as bool);
 
-  @override
-  void onDispose() => super.noSuchMethod(
-    Invocation.method(#onDispose, []),
-    returnValueForMissingStub: null,
-  );
-
   @override
   _i4.Future<void> login() =>
       (super.noSuchMethod(
@@ -95,6 +89,12 @@ class MockLoginViewModel extends _i1.Mock implements _i2.LoginViewModel {
           )
           as _i4.Future<void>);
 
+  @override
+  void onDispose() => super.noSuchMethod(
+    Invocation.method(#onDispose, []),
+    returnValueForMissingStub: null,
+  );
+
   @override
   void dispose() => super.noSuchMethod(
     Invocation.method(#dispose, []),
