diff --git a/test/app_start_test.dart b/test/app_start_test.dart
index 83d6fcb..f9c2d2b 100644
--- a/test/app_start_test.dart
+++ b/test/app_start_test.dart
@@ -67,7 +67,11 @@ void main() {
       GoRoute(
         path: '/login',
         builder: (context, state) => LoginScreen(
-          viewModel: LoginViewModel(mockAuthClient, mockAuthService, AuthViewModel(mockAuthService)),
+          viewModel: LoginViewModel(
+            mockAuthClient,
+            mockAuthService,
+            AuthViewModel(mockAuthService),
+          ),
         ),
       ),
       GoRoute(
@@ -132,7 +136,11 @@ void main() {
       GoRoute(
         path: '/login',
         builder: (context, state) => LoginScreen(
-          viewModel: LoginViewModel(mockAuthClient, mockAuthService, AuthViewModel(mockAuthService)),
+          viewModel: LoginViewModel(
+            mockAuthClient,
+            mockAuthService,
+            AuthViewModel(mockAuthService),
+          ),
         ),
       ),
       GoRoute(
