diff --git a/test/screens/login_screen/login_screen_test.dart b/test/screens/login_screen/login_screen_test.dart
index 798fd15..0559067 100644
--- a/test/screens/login_screen/login_screen_test.dart
+++ b/test/screens/login_screen/login_screen_test.dart
@@ -50,7 +50,9 @@ void main() {
       expect(find.byType(CircularProgressIndicator), findsOneWidget);
     });
 
-    testWidgets('shows error message when present', (WidgetTester tester) async {
+    testWidgets('shows error message when present', (
+      WidgetTester tester,
+    ) async {
       when(mockLoginViewModel.errorMessage).thenReturn('Login failed');
 
       await tester.pumpWidget(
