diff --git a/lib/src/init.config.dart b/lib/src/init.config.dart
index e60d2de..4141681 100644
--- a/lib/src/init.config.dart
+++ b/lib/src/init.config.dart
@@ -92,12 +92,11 @@ extension GetItInjectableX on _i174.GetIt {
       () => _i492.LoadingRouteProvider(gh<_i850.AuthViewModel>()),
       instanceName: 'LoadingRouteProvider',
     );
-    gh.lazySingletonAsync<_i25.Client>(
+    gh.lazySingleton<_i25.Client>(
       () => ferryModule.ferryClient(gh<_i564.FerryClientService>()),
     );
-    gh.factoryAsync<_i627.UserDetailsViewModelFactory>(
-      () async =>
-          _i627.UserDetailsViewModelFactory(await getAsync<_i25.Client>()),
+    gh.factory<_i627.UserDetailsViewModelFactory>(
+      () => _i627.UserDetailsViewModelFactory(gh<_i25.Client>()),
     );
     gh.factory<_i518.RouteProvider>(
       () => _i176.HomeRouteProvider(
@@ -113,9 +112,9 @@ extension GetItInjectableX on _i174.GetIt {
         gh<_i850.AuthViewModel>(),
       ),
     );
-    gh.factoryAsync<_i518.RouteProvider>(
-      () async => _i57.UserDetailsRouteProvider(
-        await getAsync<_i627.UserDetailsViewModelFactory>(),
+    gh.factory<_i518.RouteProvider>(
+      () => _i57.UserDetailsRouteProvider(
+        gh<_i627.UserDetailsViewModelFactory>(),
       ),
       instanceName: 'UserDetailsRouteProvider',
     );
