diff --git a/src/modules/build/build.module.ts b/src/modules/build/build.module.ts
index cf6b400..a6ca8a0 100644
--- a/src/modules/build/build.module.ts
+++ b/src/modules/build/build.module.ts
@@ -2,9 +2,10 @@ import { Module } from '@nestjs/common';
 
 import { BuildCommand } from './commands/build.command';
 import { InteractiveService } from './services/interactive.service';
+import { CollectionService } from './services/collection.service';
 
 @Module({
-  providers: [BuildCommand, InteractiveService],
-  exports: [BuildCommand, InteractiveService],
+  providers: [BuildCommand, InteractiveService, CollectionService],
+  exports: [BuildCommand, InteractiveService, CollectionService],
 })
 export class BuildModule {}
\ No newline at end of file
