diff --git a/.cursor/rules/components.mdc b/.cursor/rules/components.mdc
index 0dd2485..9472099 100644
--- a/.cursor/rules/components.mdc
+++ b/.cursor/rules/components.mdc
@@ -1,6 +1,9 @@
 ---
-description: rules for working with components 
+description: rules for working with components
 globs: 
 alwaysApply: false
 ---
-* When creating a brand new stand alone component (that's not a child/tightly coupled component), use the `nlx hygen component new --name [ComponentNameHere]` from package.json
\ No newline at end of file
+* When creating a brand new stand alone component (that's not a child/tightly coupled component), use the `nlx hygen component new --name [ComponentNameHere]` from package.json* When creating template refs always name them with the `El` suffix. For example:
+  * `const textareaEl = useTemplateRef('textareaEl')`  
+  * Always use `useTemplateRef('[ref identifier here]')` to define template refs (not `ref()`)
+  * ShadCN has components like `Textarea` that seem like native components but are not. Remember to account for that.
