diff --git a/src/components/CanvasContainer.tsx b/src/components/CanvasContainer.tsx
new file mode 100644
index 0000000..dc88c09
--- /dev/null
+++ b/src/components/CanvasContainer.tsx
@@ -0,0 +1,13 @@
+import Canvas from "./Canvas";
+import CanvasToolbar from "./CanvasToolbar";
+
+export default function CanvasContainer() {
+  return (
+    <div className="flex h-full w-full flex-col">
+      <div className="flex-1 size-full overflow-auto bg-gray-100">
+        <Canvas />
+      </div>
+      <CanvasToolbar />
+    </div>
+  );
+}
