diff --git a/stories/ReportsPage.stories.ts b/stories/ReportsPage.stories.ts
index bf54723..2209281 100644
--- a/stories/ReportsPage.stories.ts
+++ b/stories/ReportsPage.stories.ts
@@ -21,8 +21,6 @@ const meta: Meta<typeof ReportsPage> = {
     },
     tags: ['autodocs'],
     argTypes: {
-        onReportSelect: { action: 'report-select' },
-        onReportDelete: { action: 'report-delete' },
         onReportDownload: { action: 'report-download' },
     },
 };
