diff --git a/src/router/index.ts b/src/router/index.ts
index 7310ae9..562dbfd 100644
--- a/src/router/index.ts
+++ b/src/router/index.ts
@@ -1,6 +1,6 @@
 import { createRouter, createWebHistory } from 'vue-router';
 
-import HomeView from '../views/home-view.vue';
+import HomeView from '@/views/home-view.vue';
 
 const router = createRouter({
 	history: createWebHistory(import.meta.env.BASE_URL),
