diff --git a/client/src/views/AuthView.vue b/client/src/views/AuthView.vue
index fd21a2f..b227999 100644
--- a/client/src/views/AuthView.vue
+++ b/client/src/views/AuthView.vue
@@ -51,8 +51,8 @@ import { useRouter } from 'vue-router'
 const gameStore = useGameStore()
 const router = useRouter()
 const isLogin = ref(true)
-const username = ref('test')
-const password = ref('test')
+const username = ref('')
+const password = ref('')
 const error = ref('')
 
 // Store listener IDs
