diff --git a/src/data/effectProcessor.js b/src/data/effectProcessor.js
index a3461a5..7fd6947 100644
--- a/src/data/effectProcessor.js
+++ b/src/data/effectProcessor.js
@@ -4,7 +4,7 @@
 import effectDescriptions from '../data/effectDescription.js';
 import { dealDamage } from './battleUtils.js';
 import { addEffectLog, addBattleLog } from './battleLogUtils.js';
-import {enqueueDelay} from "./animationDispatcher";
+import {enqueueDelay} from "./animationInstructionHelpers.js";
 
 /**
  * 处理回合开始时触发的效果
