diff --git a/src/data/enemies/boss.js b/src/data/enemies/boss.js
index 79ca365..804e6c3 100644
--- a/src/data/enemies/boss.js
+++ b/src/data/enemies/boss.js
@@ -1,7 +1,7 @@
 import Enemy from '../enemy.js';
 import { launchAttack } from '../battleUtils.js';
 import { addEnemyActionLog } from '../battleLogUtils.js';
-import {enqueueDelay} from "../animationDispatcher";
+import {enqueueDelay} from "../animationInstructionHelpers";
 
 // MEFM-3 Boss敌人
 export class MEFM3 extends Enemy {
