diff --git a/test/screens/loading_screen/loading_route_provider_test.dart b/test/screens/loading_screen/loading_route_provider_test.dart
index 9c8dc7f..977615a 100644
--- a/test/screens/loading_screen/loading_route_provider_test.dart
+++ b/test/screens/loading_screen/loading_route_provider_test.dart
@@ -1,5 +1,4 @@
 import 'package:flutter_test/flutter_test.dart';
-import 'package:mockito/mockito.dart';
 import 'package:mockito/annotations.dart';
 import 'package:go_router/go_router.dart';
 import 'package:gh3/src/screens/loading_screen/loading_route_provider.dart';
