diff --git a/app/components/ui/alert-dialog/AlertDialogCancel.vue b/app/components/ui/alert-dialog/AlertDialogCancel.vue
index f9cba4e..634fc9f 100644
--- a/app/components/ui/alert-dialog/AlertDialogCancel.vue
+++ b/app/components/ui/alert-dialog/AlertDialogCancel.vue
@@ -1,23 +1,23 @@
 <script setup lang="ts">
-import type { HTMLAttributes } from 'vue'
-import { reactiveOmit } from '@vueuse/core'
-import { AlertDialogCancel, type AlertDialogCancelProps } from 'reka-ui'
-import { cn } from '@/lib/utils'
-import { buttonVariants } from '@/components/ui/button'
+import type { HTMLAttributes } from "vue";
+import { reactiveOmit } from "@vueuse/core";
+import { AlertDialogCancel, type AlertDialogCancelProps } from "reka-ui";
+import { cn } from "@/lib/utils";
+import { buttonVariants } from "~/components/Ui/button";
 
-const props = defineProps<AlertDialogCancelProps & { class?: HTMLAttributes['class'] }>()
+const props = defineProps<
+  AlertDialogCancelProps & { class?: HTMLAttributes["class"] }
+>();
 
-const delegatedProps = reactiveOmit(props, 'class')
+const delegatedProps = reactiveOmit(props, "class");
 </script>
 
 <template>
   <AlertDialogCancel
     v-bind="delegatedProps"
-    :class="cn(
-      buttonVariants({ variant: 'outline' }),
-      'mt-2 sm:mt-0',
-      props.class,
-    )"
+    :class="
+      cn(buttonVariants({ variant: 'outline' }), 'mt-2 sm:mt-0', props.class)
+    "
   >
     <slot />
   </AlertDialogCancel>
