diff --git a/app/components/ui/form/FormLabel.vue b/app/components/ui/form/FormLabel.vue
index a7d69bb..dfd4aeb 100644
--- a/app/components/ui/form/FormLabel.vue
+++ b/app/components/ui/form/FormLabel.vue
@@ -1,23 +1,20 @@
 <script lang="ts" setup>
-import type { LabelProps } from 'reka-ui'
-import type { HTMLAttributes } from 'vue'
-import { cn } from '@/lib/utils'
-import { Label } from '@/components/ui/label'
-import { useFormField } from './useFormField'
+import type { LabelProps } from "reka-ui";
+import type { HTMLAttributes } from "vue";
+import { cn } from "@/lib/utils";
+import { Label } from "~/components/Ui/label";
+import { useFormField } from "./useFormField";
 
-const props = defineProps<LabelProps & { class?: HTMLAttributes['class'] }>()
+const props = defineProps<LabelProps & { class?: HTMLAttributes["class"] }>();
 
-const { error, formItemId } = useFormField()
+const { error, formItemId } = useFormField();
 </script>
 
 <template>
   <Label
     data-slot="form-label"
     :data-error="!!error"
-    :class="cn(
-      'data-[error=true]:text-destructive-foreground',
-      props.class,
-    )"
+    :class="cn('data-[error=true]:text-destructive-foreground', props.class)"
     :for="formItemId"
   >
     <slot />
