diff --git a/app/components/ui/pagination/PaginationNext.vue b/app/components/ui/pagination/PaginationNext.vue
index 7c4ce50..961643d 100644
--- a/app/components/ui/pagination/PaginationNext.vue
+++ b/app/components/ui/pagination/PaginationNext.vue
@@ -1,27 +1,38 @@
 <script setup lang="ts">
-import type { PaginationNextProps } from 'reka-ui'
-import type { HTMLAttributes } from 'vue'
-import { reactiveOmit } from '@vueuse/core'
-import { ChevronRightIcon } from 'lucide-vue-next'
-import { PaginationNext, useForwardProps } from 'reka-ui'
-import { cn } from '@/lib/utils'
-import { buttonVariants, type ButtonVariants } from '@/components/ui/button'
+import type { PaginationNextProps } from "reka-ui";
+import type { HTMLAttributes } from "vue";
+import { reactiveOmit } from "@vueuse/core";
+import { ChevronRightIcon } from "lucide-vue-next";
+import { PaginationNext, useForwardProps } from "reka-ui";
+import { cn } from "@/lib/utils";
+import { buttonVariants, type ButtonVariants } from "~/components/Ui/button";
 
-const props = withDefaults(defineProps<PaginationNextProps & {
-  size?: ButtonVariants['size']
-  class?: HTMLAttributes['class']
-}>(), {
-  size: 'default',
-})
+const props = withDefaults(
+  defineProps<
+    PaginationNextProps & {
+      size?: ButtonVariants["size"];
+      class?: HTMLAttributes["class"];
+    }
+  >(),
+  {
+    size: "default",
+  }
+);
 
-const delegatedProps = reactiveOmit(props, 'class', 'size')
-const forwarded = useForwardProps(delegatedProps)
+const delegatedProps = reactiveOmit(props, "class", "size");
+const forwarded = useForwardProps(delegatedProps);
 </script>
 
 <template>
   <PaginationNext
     data-slot="pagination-next"
-    :class="cn(buttonVariants({ variant: 'ghost', size }), 'gap-1 px-2.5 sm:pr-2.5', props.class)"
+    :class="
+      cn(
+        buttonVariants({ variant: 'ghost', size }),
+        'gap-1 px-2.5 sm:pr-2.5',
+        props.class
+      )
+    "
     v-bind="forwarded"
   >
     <slot>
