diff --git a/app/components/ui/range-calendar/RangeCalendarPrevButton.vue b/app/components/ui/range-calendar/RangeCalendarPrevButton.vue
index 9cc1df1..7e51779 100644
--- a/app/components/ui/range-calendar/RangeCalendarPrevButton.vue
+++ b/app/components/ui/range-calendar/RangeCalendarPrevButton.vue
@@ -1,27 +1,35 @@
 <script lang="ts" setup>
-import type { HTMLAttributes } from 'vue'
-import { reactiveOmit } from '@vueuse/core'
-import { ChevronLeft } from 'lucide-vue-next'
-import { RangeCalendarPrev, type RangeCalendarPrevProps, useForwardProps } from 'reka-ui'
-import { cn } from '@/lib/utils'
-import { buttonVariants } from '@/components/ui/button'
+import type { HTMLAttributes } from "vue";
+import { reactiveOmit } from "@vueuse/core";
+import { ChevronLeft } from "lucide-vue-next";
+import {
+  RangeCalendarPrev,
+  type RangeCalendarPrevProps,
+  useForwardProps,
+} from "reka-ui";
+import { cn } from "@/lib/utils";
+import { buttonVariants } from "~/components/Ui/button";
 
-const props = defineProps<RangeCalendarPrevProps & { class?: HTMLAttributes['class'] }>()
+const props = defineProps<
+  RangeCalendarPrevProps & { class?: HTMLAttributes["class"] }
+>();
 
-const delegatedProps = reactiveOmit(props, 'class')
+const delegatedProps = reactiveOmit(props, "class");
 
-const forwardedProps = useForwardProps(delegatedProps)
+const forwardedProps = useForwardProps(delegatedProps);
 </script>
 
 <template>
   <RangeCalendarPrev
     data-slot="range-calendar-prev-button"
-    :class="cn(
-      buttonVariants({ variant: 'outline' }),
-      'absolute left-1',
-      'size-7 bg-transparent p-0 opacity-50 hover:opacity-100',
-      props.class,
-    )"
+    :class="
+      cn(
+        buttonVariants({ variant: 'outline' }),
+        'absolute left-1',
+        'size-7 bg-transparent p-0 opacity-50 hover:opacity-100',
+        props.class
+      )
+    "
     v-bind="forwardedProps"
   >
     <slot>
