diff --git a/app/components/ui/sidebar/SidebarInput.vue b/app/components/ui/sidebar/SidebarInput.vue
index ecaa977..b20e37d 100644
--- a/app/components/ui/sidebar/SidebarInput.vue
+++ b/app/components/ui/sidebar/SidebarInput.vue
@@ -1,21 +1,18 @@
 <script setup lang="ts">
-import type { HTMLAttributes } from 'vue'
-import { cn } from '@/lib/utils'
-import { Input } from '@/components/ui/input'
+import type { HTMLAttributes } from "vue";
+import { cn } from "@/lib/utils";
+import { Input } from "~/components/Ui/input";
 
 const props = defineProps<{
-  class?: HTMLAttributes['class']
-}>()
+  class?: HTMLAttributes["class"];
+}>();
 </script>
 
 <template>
   <Input
     data-slot="sidebar-input"
     data-sidebar="input"
-    :class="cn(
-      'bg-background h-8 w-full shadow-none',
-      props.class,
-    )"
+    :class="cn('bg-background h-8 w-full shadow-none', props.class)"
   >
     <slot />
   </Input>
