diff --git a/app/components/ui/sidebar/SidebarMenuSkeleton.vue b/app/components/ui/sidebar/SidebarMenuSkeleton.vue
index 6302c35..ef86eaf 100644
--- a/app/components/ui/sidebar/SidebarMenuSkeleton.vue
+++ b/app/components/ui/sidebar/SidebarMenuSkeleton.vue
@@ -1,16 +1,16 @@
 <script setup lang="ts">
-import { computed, type HTMLAttributes } from 'vue'
-import { cn } from '@/lib/utils'
-import { Skeleton } from '@/components/ui/skeleton'
+import { computed, type HTMLAttributes } from "vue";
+import { cn } from "@/lib/utils";
+import { Skeleton } from "~/components/Ui/skeleton";
 
 const props = defineProps<{
-  showIcon?: boolean
-  class?: HTMLAttributes['class']
-}>()
+  showIcon?: boolean;
+  class?: HTMLAttributes["class"];
+}>();
 
 const width = computed(() => {
-  return `${Math.floor(Math.random() * 40) + 50}%`
-})
+  return `${Math.floor(Math.random() * 40) + 50}%`;
+});
 </script>
 
 <template>
