diff --git a/app/components/ui/sidebar/SidebarSeparator.vue b/app/components/ui/sidebar/SidebarSeparator.vue
index 1803bff..a949264 100644
--- a/app/components/ui/sidebar/SidebarSeparator.vue
+++ b/app/components/ui/sidebar/SidebarSeparator.vue
@@ -1,11 +1,11 @@
 <script setup lang="ts">
-import type { HTMLAttributes } from 'vue'
-import { cn } from '@/lib/utils'
-import { Separator } from '@/components/ui/separator'
+import type { HTMLAttributes } from "vue";
+import { cn } from "@/lib/utils";
+import { Separator } from "~/components/Ui/separator";
 
 const props = defineProps<{
-  class?: HTMLAttributes['class']
-}>()
+  class?: HTMLAttributes["class"];
+}>();
 </script>
 
 <template>
