diff --git a/app/components/ui/sidebar/SidebarTrigger.vue b/app/components/ui/sidebar/SidebarTrigger.vue
index 59f29b9..5acc3b7 100644
--- a/app/components/ui/sidebar/SidebarTrigger.vue
+++ b/app/components/ui/sidebar/SidebarTrigger.vue
@@ -1,15 +1,15 @@
 <script setup lang="ts">
-import type { HTMLAttributes } from 'vue'
-import { PanelLeft } from 'lucide-vue-next'
-import { cn } from '@/lib/utils'
-import { Button } from '@/components/ui/button'
-import { useSidebar } from './utils'
+import type { HTMLAttributes } from "vue";
+import { PanelLeft } from "lucide-vue-next";
+import { cn } from "@/lib/utils";
+import { Button } from "~/components/Ui/button";
+import { useSidebar } from "./utils";
 
 const props = defineProps<{
-  class?: HTMLAttributes['class']
-}>()
+  class?: HTMLAttributes["class"];
+}>();
 
-const { toggleSidebar } = useSidebar()
+const { toggleSidebar } = useSidebar();
 </script>
 
 <template>
