diff --git a/nuxt.config.ts b/nuxt.config.ts
index 172c514..21f4049 100644
--- a/nuxt.config.ts
+++ b/nuxt.config.ts
@@ -16,17 +16,18 @@ export default defineNuxtConfig({
     "shadcn-nuxt",
     "@nuxt/test-utils/module",
     "@nuxt/icon",
+    "@nuxt/image",
   ],
   shadcn: {
     /**
      * Prefix for all the imported component
      */
-    prefix: "",
+    prefix: "Ui",
     /**
      * Directory that the component lives in.
-     * @default "./components/ui"
+     * @default "./components/Ui"
      */
-    componentDir: "./app/components/ui",
+    componentDir: "./app/components/Ui",
   },
   nitro: {
     experimental: {
@@ -41,4 +42,4 @@ export default defineNuxtConfig({
       },
     },
   },
-});
+});
\ No newline at end of file
