diff --git a/.cursor/rules/components.mdc b/.cursor/rules/components.mdc
index 16683ff..39896c6 100644
--- a/.cursor/rules/components.mdc
+++ b/.cursor/rules/components.mdc
@@ -3,6 +3,7 @@ description: rules for working with components
 globs: 
 alwaysApply: false
 ---
+
 ## General Component Rules
 
 - Shad-cn Vue is installed and all of it's components are stored in `/components/Ui`. They can be for most common UI elements.
@@ -64,4 +65,22 @@ app/components/
   - `const myComponentRef = useTemplateRef('myComponentRef')`   
 - Always use `useTemplateRef('[ref identifier here]')` to define template refs (not `ref()`)
 
+## Icons
+
+- Nuxt Icon is installed. 
+- Display an icon with the icon component. For example:
+  - `<Icon name="lucide:refresh">
+- Rember to prefix the icon with the packname (like `lucide:`)  
+- Any iconify icon can be used
+
+# Style
+
+- Primarily use shadcn classes for color styles. For example:
+  - bg-primary
+  - text-primary-foreground
+  - bg-muted
+  - text-muted-foreground
 
+- Pair bg and text-foreground colors appropriately. For example:
+  - bg-primary and text-primary-foreground
+  - bg-muted and text-muted-foreground
\ No newline at end of file
