diff --git a/.cursor/rules/wf-comp.mdc b/.cursor/rules/wf-comp.mdc
new file mode 100644
index 0000000..5db4b06
--- /dev/null
+++ b/.cursor/rules/wf-comp.mdc
@@ -0,0 +1,9 @@
+---
+description: 
+globs: 
+alwaysApply: false
+---
+* Create simple props down, events up components that don't interact with any API endpoints.
+* Optimize them as if their consuming components WILL be getting data from API endpoints
+* Add a page in the `app/pages/playground` directory with hardcoded dummy data passed in to the props and reacting to events.
+* ALWAYS read the [components.mdc](mdc:.cursor/rules/components.mdc)
\ No newline at end of file
