diff --git a/lib/main.dart b/lib/main.dart
index 2ffb8a2..5af8f1a 100644
--- a/lib/main.dart
+++ b/lib/main.dart
@@ -1,6 +1,5 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/foundation.dart';
-import 'package:flutter/services.dart';
 import 'package:gh3/src/init.dart';
 import 'package:gh3/src/screens/app/gh3_app.dart';
 import 'package:gh3/src/screens/app/auth_viewmodel.dart';
@@ -16,7 +15,7 @@ Future<void> main() async {
       if (kDebugMode) {
         enableFlutterDriverExtension();
       }
-      
+
       WidgetsFlutterBinding.ensureInitialized();
 
       configureDependencies();
