diff --git a/worker/src/utils.rs b/worker/src/utils.rs
deleted file mode 100644
index 7c1b13c..0000000
--- a/worker/src/utils.rs
+++ /dev/null
@@ -1,9 +0,0 @@
-pub fn set_panic_hook() {
-    // When the `console_error_panic_hook` feature is enabled, we can call the
-    // `set_panic_hook` function at least once during initialization, and then
-    // we will get better error messages if our code ever panics.
-    //
-    // For more details see
-    // https://github.com/rustwasm/console_error_panic_hook#readme
-    console_error_panic_hook::set_once();
-}
\ No newline at end of file
