diff --git a/vite.config.ts b/vite.config.ts
index 933899e..d6dffa8 100644
--- a/vite.config.ts
+++ b/vite.config.ts
@@ -1,5 +1,5 @@
 import tailwindcss from '@tailwindcss/vite';
-import { TanStackRouterVite } from '@tanstack/router-plugin/vite';
+import { tanstackRouter } from '@tanstack/router-plugin/vite';
 import react from '@vitejs/plugin-react';
 import { defineConfig } from 'vite';
 import tsconfigPaths from 'vite-tsconfig-paths';
@@ -7,14 +7,14 @@ import tsconfigPaths from 'vite-tsconfig-paths';
 // https://vite.dev/config/
 export default defineConfig({
   plugins: [
-    TanStackRouterVite({
+    react(),
+    tailwindcss(),
+    tanstackRouter({
       target: 'react',
       autoCodeSplitting: true,
       routesDirectory: './src/app/routes',
       generatedRouteTree: './src/shared/utils/router/routeTree.gen.ts',
     }),
-    react(),
-    tailwindcss(),
     tsconfigPaths(),
   ],
 });
