diff --git a/lib/main.dart b/lib/main.dart
index 2148c0a..1f9e7b1 100644
--- a/lib/main.dart
+++ b/lib/main.dart
@@ -4,6 +4,7 @@ import 'package:gh3/src/init.dart';
 import 'package:gh3/src/screens/app/gh3_app.dart';
 import 'package:gh3/src/screens/home_screen/home_viewmodel.dart';
 import 'package:gh3/src/screens/app/auth_viewmodel.dart';
+import 'package:gh3/src/screens/user_details/user_details_viewmodel_factory.dart';
 import 'package:gh3/src/services/auth_service.dart';
 import 'package:gh3/src/services/github_auth_client.dart';
 
@@ -15,6 +16,7 @@ Future<void> main() async {
   final authService = getIt<AuthService>();
   final githubAuthClient = getIt<GithubAuthClient>();
   final homeViewModel = await GetIt.instance.getAsync<HomeViewModel>();
+  final userDetailsViewModelFactory = getIt<UserDetailsViewModelFactory>();
 
   // Create ViewModels manually with their dependencies
   final authVM = AuthViewModel(authService);
@@ -26,6 +28,7 @@ Future<void> main() async {
       authService: authService,
       githubAuthClient: githubAuthClient,
       homeViewModel: homeViewModel,
+      userDetailsViewModelFactory: userDetailsViewModelFactory,
     ),
   );
 }
