diff --git a/lib/src/screens/home_screen/home_viewmodel.dart b/lib/src/screens/home_screen/home_viewmodel.dart
index 9e403dc..8f5a7f8 100644
--- a/lib/src/screens/home_screen/home_viewmodel.dart
+++ b/lib/src/screens/home_screen/home_viewmodel.dart
@@ -4,9 +4,7 @@ import 'package:ferry/ferry.dart';
 import '__generated__/home_viewmodel.req.gql.dart';
 import '__generated__/home_viewmodel.data.gql.dart';
 import '__generated__/home_viewmodel.var.gql.dart';
-import 'package:injectable/injectable.dart';
 
-@injectable
 class HomeViewModel extends ChangeNotifier {
   final Client _ferryClient;
 
