diff --git a/test/app_start_test.mocks.dart b/test/app_start_test.mocks.dart
index 237d746..26de0fe 100644
--- a/test/app_start_test.mocks.dart
+++ b/test/app_start_test.mocks.dart
@@ -7,11 +7,13 @@ import 'dart:async' as _i3;
 
 import 'package:ferry/ferry.dart' as _i5;
 import 'package:ferry/src/fetch_policy_typed_link.dart' as _i2;
+import 'package:gh3/src/routing/route_collection_service.dart' as _i12;
 import 'package:gh3/src/services/auth_service.dart' as _i11;
 import 'package:gh3/src/services/github_auth_client.dart' as _i4;
 import 'package:gh3/src/services/scope_service.dart' as _i9;
 import 'package:gh3/src/services/timer_service.dart' as _i10;
 import 'package:gh3/src/services/token_storage.dart' as _i8;
+import 'package:go_router/go_router.dart' as _i13;
 import 'package:gql/ast.dart' as _i6;
 import 'package:mockito/mockito.dart' as _i1;
 import 'package:mockito/src/dummies.dart' as _i7;
@@ -402,3 +404,18 @@ class MockAuthService extends _i1.Mock implements _i11.AuthService {
           )
           as _i3.Future<void>);
 }
+
+/// A class which mocks [RouteCollectionService].
+///
+/// See the documentation for Mockito's code generation for more information.
+class MockRouteCollectionService extends _i1.Mock
+    implements _i12.RouteCollectionService {
+  @override
+  List<_i13.RouteBase> collectRoutes() =>
+      (super.noSuchMethod(
+            Invocation.method(#collectRoutes, []),
+            returnValue: <_i13.RouteBase>[],
+            returnValueForMissingStub: <_i13.RouteBase>[],
+          )
+          as List<_i13.RouteBase>);
+}
