diff --git a/dicomcriterion/criterion.py b/dicomcriterion/criterion.py
index dc2d288..5a0ae2e 100644
--- a/dicomcriterion/criterion.py
+++ b/dicomcriterion/criterion.py
@@ -192,7 +192,9 @@ class Criterion:
                 else:
                     # This shouldn't happen if our implementation is correct
                     raise ValueError(
-                        f"Symbol {symbol_key} not found in " f"symbol mapping"
+                        f"Symbol {symbol_key} not found "
+                        f"in "
+                        f"symbol mapping"  # noqa E713
                     )
 
             # Evaluate the boolean expression using the symbol values
