diff --git a/.cursor/rules/api.mdc b/.cursor/rules/api.mdc
index fd1b9c0..797e37b 100644
--- a/.cursor/rules/api.mdc
+++ b/.cursor/rules/api.mdc
@@ -4,18 +4,8 @@ globs: server/api/**/*.ts
 alwaysApply: false
 ---
 * All API endpoints follow the OpenAPI specification.
-* Document all endpoints with at the top of the files (after imports) like this:
-
-```typescript
-defineRouteMeta({
-  openAPI: {
-    tags: ["test"],
-    description: "Test route description",
-    parameters: [{ in: "query", name: "test", required: true }],
-  },
-});
-
-export default defineEventHandler(async () => {
- // The endpoint logic here
-});
-```
+* When creating new resources (like users, posts, etc) use the following command to bootsrap all CRUD API endpoints: `npx hygen resource new --name [singular-resource-name-here]`
+* When creating one-off API endpoints (not full resources) bootstrap the API file with the command: `npx hygen api new --path [path-relative to server/api] --method [get, post, put, delete]`
+* If you need to create an API endpoint that supports streaming:
+  *  Don't use one of the hygen commands
+  *  DO use the Nuxt MCP for reference
