diff --git a/.cursor/rules/testing.mdc b/.cursor/rules/testing.mdc
index d23a9b2..b4afdb9 100644
--- a/.cursor/rules/testing.mdc
+++ b/.cursor/rules/testing.mdc
@@ -4,4 +4,12 @@ globs:
 alwaysApply: false
 ---
 * If ever in doubt, use context7 mcp or search the web to help write a unit test
-* More tests documentation found in [README.md](mdc:tests/README.md).(tests/README.md)
\ No newline at end of file
+* More tests documentation found in [README.md](mdc:tests/README.md).(tests/README.md)
+
+## Unit Tests
+* Unit tests are written with Vitest
+* Are written alongside the code they are testing (in the same directory)
+
+## E2E Tests
+* E2E tests are written with Playwright
+* E2E tests are written in the tests/e2e/ directory
