diff --git a/nuxt.config.ts b/nuxt.config.ts
index 57ca499..172c514 100644
--- a/nuxt.config.ts
+++ b/nuxt.config.ts
@@ -16,7 +16,6 @@ export default defineNuxtConfig({
     "shadcn-nuxt",
     "@nuxt/test-utils/module",
     "@nuxt/icon",
-    "@nuxtjs/mdc",
   ],
   shadcn: {
     /**
@@ -42,4 +41,4 @@ export default defineNuxtConfig({
       },
     },
   },
-});
\ No newline at end of file
+});
