diff --git a/src/components/select-dropdown.vue b/src/components/select-dropdown.vue
index 2e32aca..d48f23b 100644
--- a/src/components/select-dropdown.vue
+++ b/src/components/select-dropdown.vue
@@ -69,7 +69,7 @@ const currentLabel = computed(() => {
 		<Select.Control>
 			<Select.Trigger
 				class="text-sm text-gray-700 font-medium px-3.5 py-2 border border-gray-300 rounded-lg bg-white flex w-full cursor-pointer transition-all items-center justify-between dark:text-gray-200 focus:outline-none dark:border-gray-600 focus:border-primary-500 hover:border-primary-400 dark:bg-gray-800 focus:ring-3 focus:ring-primary-500/20 dark:hover:border-primary-500">
-				<Select.ValueText placeholder="Select an option" class="flex-grow">
+				<Select.ValueText placeholder="Select an option" class="text-left flex-grow">
 					{{ currentLabel }}
 				</Select.ValueText>
 				<Select.Indicator class="text-gray-500 ml-2 flex-shrink-0 dark:text-gray-400">
@@ -86,8 +86,10 @@ const currentLabel = computed(() => {
 							v-for="option in properties.options"
 							:key="option.value"
 							:item="option.value"
+							@click.stop
+							@mousedown.stop
 							class="text-sm text-gray-700 font-medium px-4 py-2.5 flex cursor-pointer items-center justify-between dark:text-gray-200 hover:bg-primary-50 dark:hover:bg-primary-900/20">
-							<Select.ItemText>{{ option.label }}</Select.ItemText>
+							<Select.ItemText class="text-left">{{ option.label }}</Select.ItemText>
 							<Select.ItemIndicator class="text-primary-600 ml-2 dark:text-primary-400">
 								<div class="i-ri-check-line h-5 w-5"></div>
 							</Select.ItemIndicator>
