diff --git a/_templates/resource/new/test.ts.ejs.t b/_templates/resource/new/test.ts.ejs.t
deleted file mode 100644
index ec02a51..0000000
--- a/_templates/resource/new/test.ts.ejs.t
+++ /dev/null
@@ -1,19 +0,0 @@
----
-to: tests/e2e/api-<%= plural %>.spec.ts
----
-import { test, expect } from '@nuxt/test-utils/playwright'
-
-test.describe('/<%= plural %> API endpoints', () => {
-  let _created<%= Name %>Id: string
-
-  test('GET /<%= plural %> - should list <%= plural %>', async ({ page, goto }) => {
-    await goto('/', { waitUntil: 'hydration' })
-
-    const response = await page.request.get('/api/<%= plural %>')
-    expect(response.status()).toBe(200)
-    // TODO: Add tests for the response data
-  })
-
-  // TODO: Add tests for all the endpoints
-
-}) 
\ No newline at end of file
