diff --git a/_templates/resource/new/index.get.test.ts.ejs.t b/_templates/resource/new/index.get.test.ts.ejs.t
index 5c2e6bd..77f97ff 100644
--- a/_templates/resource/new/index.get.test.ts.ejs.t
+++ b/_templates/resource/new/index.get.test.ts.ejs.t
@@ -5,7 +5,7 @@ to: server/api/<%= plural %>/index.get.test.ts
 import { describe, it, expect } from "vitest";
 
 describe("GET /api/<%= plural %>", () => {
-  it("should return the standard payload shape prescribed by defineApiResponse", async () => {
+  it("should return the standard payload shape prescribed by defineApiPaginatedResponse", async () => {
     const response = await $vitestFetch("/api/<%= plural %>");
 
     expect(response.statusCode).toBe(200);
@@ -13,6 +13,12 @@ describe("GET /api/<%= plural %>", () => {
     expect(response).toHaveProperty("statusMessage");
     expect(response).toHaveProperty("data");
     expect(response).toHaveProperty("url");
+    expect(response).toHaveProperty("pagination");
+    expect(response.pagination).toHaveProperty("page");
+    expect(response.pagination).toHaveProperty("limit");
+    expect(response.pagination).toHaveProperty("total");
+    expect(response.pagination).toHaveProperty("currentPageUrl");
+    expect(response.pagination).toHaveProperty("firstPageUrl");
     expect(response.error).toBeFalsy();
   });
 
@@ -64,5 +70,7 @@ describe("GET /api/<%= plural %>", () => {
     expect(response.pagination.limit).toBe(20);
   });
 
-  // TODO: Add tests for edge cases (empty results, invalid pagination, etc.)
+  // TODO: Add tests for edge cases (empty results, invalid pagination, etc.) 
+  // TODO: Add tests for other business logic
+    // Remember to handle edge cases and error scenarios
 }); 
\ No newline at end of file
