diff --git a/game/tictactoe.go b/game/tictactoe.go
index e3d1e63..346017a 100644
--- a/game/tictactoe.go
+++ b/game/tictactoe.go
@@ -1,12 +1,23 @@
 package game
 
+import (
+	"fmt"
+)
+
 type Game struct {
 	board [9]string
 	text  string
 }
 
-func (g *Game) MakeMove(position int) {
+func (g *Game) MakeMove(position int) error {
+	if position < 0 || position > 8 {
+		return fmt.Errorf("invalid position")
+	}
+	if g.Board()[position] != "" {
+		return fmt.Errorf("cell already occupied")
+	}
 	g.board[position] = "X"
+	return nil
 }
 
 func NewGame() *Game {
