diff --git a/src/components/molecules/radar-chart.tsx b/src/components/molecules/radar-chart.tsx
index 95c94bc..7767395 100644
--- a/src/components/molecules/radar-chart.tsx
+++ b/src/components/molecules/radar-chart.tsx
@@ -7,7 +7,6 @@ import {
   PolarRadiusAxis,
   Radar,
   ResponsiveContainer,
-  Text,
 } from 'recharts';
 import { cn } from '@/lib/utils';
 import { getStatusFromScore } from '@/lib/utils/scores';
