diff --git a/src/lib/constants.ts b/src/lib/constants.ts
index dd0a25b..f85cf50 100644
--- a/src/lib/constants.ts
+++ b/src/lib/constants.ts
@@ -1,4 +1,4 @@
-export const CANVAS_ROWS = 40; // Num of rows in the canvas
-export const CANVAS_COLS = 100; // Num of columns in the canvas
-export const CELL_WIDTH = 5; // Width of each cell in pixels
-export const CELL_HEIGHT = 8; // Height of each cell in pixels
+export const CANVAS_ROWS = 45; // Num of rows in the canvas
+export const CANVAS_COLS = 80; // Num of columns in the canvas
+export const CELL_WIDTH = 10; // Width of each cell in pixels
+export const CELL_HEIGHT = 12; // Height of each cell in pixels
