diff --git a/src/components/ActionPanel.vue b/src/components/ActionPanel.vue
index 478c321..78acc70 100644
--- a/src/components/ActionPanel.vue
+++ b/src/components/ActionPanel.vue
@@ -76,10 +76,10 @@ export default {
   },
   methods: {
     onEndTurnButtonClicked() {
-      backendEventBus.emit(EventNames.Player.END_TURN);
+      backendEventBus.emit(EventNames.Battle.PLAYER_END_TURN);
     },
     onDropSkillButtonClicked() {
-      backendEventBus.emit(EventNames.Player.DROP_SKILL);
+      backendEventBus.emit(EventNames.Battle.PLAYER_DROP_SKILL);
     },
     onDeckClick() {
       // 打开牌库覆盖面板，并隐藏悬浮tooltip
