diff --git a/src/components/SkillsHand.vue b/src/components/SkillsHand.vue
index c68d4de..607d000 100644
--- a/src/components/SkillsHand.vue
+++ b/src/components/SkillsHand.vue
@@ -238,7 +238,7 @@ export default {
         const actionPointCost = skill.actionPointCost;
         const mouseX = event.clientX;
         const mouseY = event.clientY;
-        backendEventBus.emit(EventNames.Player.USE_SKILL, skill.uniqueID);
+        backendEventBus.emit(EventNames.Battle.PLAYER_USE_SKILL, skill.uniqueID);
         this.generateParticleEffects(manaCost, actionPointCost, mouseX, mouseY);
       }
     },
