diff --git a/src/data/effectDescription.js b/src/data/effectDescription.js
index 976e377..08b406c 100644
--- a/src/data/effectDescription.js
+++ b/src/data/effectDescription.js
@@ -35,6 +35,13 @@ const effectDescriptions = {
     type: 'debuff',
     icon: '🩶'
   },
+  '火焰抗性': {
+    name: '火焰抗性',
+    description: '减少层数点受到的燃烧伤害',
+    type: 'buff',
+    icon: '🔥',
+    color: '#FF8C00' // 深橙色
+  },
 
   // 回合开始时触发的效果
   '泉涌': {
@@ -51,6 +58,13 @@ const effectDescriptions = {
     icon: '💨',
     color: '#808080' // 灰色
   },
+  '吸热': {
+    name: '吸热',
+    description: '回合开始时，减少层数层燃烧',
+    type: 'buff',
+    icon: '❄️',
+    color: '#1E90FF' // 道奇蓝
+  },
   '燃烧': {
     name: '燃烧',
     description: '回合开始时，受到层数点伤害，层数减1',
