diff --git a/backend/src/utils/swagger.ts b/backend/src/utils/swagger.ts
index 09ef447..6f8d72e 100644
--- a/backend/src/utils/swagger.ts
+++ b/backend/src/utils/swagger.ts
@@ -136,7 +136,7 @@ const options: swaggerJsdoc.Options = {
       },
     },
   },
-  apis: ['./src/routes/*.ts', './src/controllers/*.ts'],
+  apis: ['./dist/routes/*.js', './dist/controllers/*.js', './src/routes/*.ts', './src/controllers/*.ts'],
 };
 
 const specs = swaggerJsdoc(options);
