diff --git a/src/BallDragDrop/BallDragDrop.csproj b/src/BallDragDrop/BallDragDrop.csproj
index a5a6f29..7b0dbcb 100644
--- a/src/BallDragDrop/BallDragDrop.csproj
+++ b/src/BallDragDrop/BallDragDrop.csproj
@@ -47,6 +47,9 @@
     <None Update="Resources\Cursors\*.png">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="appsettings.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
     <None Update="log4net.config">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
