diff --git a/.cursor/rules/database.mdc b/.cursor/rules/database.mdc
index 7850774..9f85ca2 100644
--- a/.cursor/rules/database.mdc
+++ b/.cursor/rules/database.mdc
@@ -7,10 +7,12 @@ alwaysApply: false
 * Project Schema - [drizzle-schema.ts](mdc:server/db/drizzle-schema.ts)
 * Changes to the schema should be made in the schema file and then run `npx drizzle-kit push` to update the database.
 * On the server side, call the auto-imported `useDb()` function
+* On the server side, call the auto-imported `useDbSchema()` function
 * Seeds should be created as nitro tasks:
 
 ```ts
 // server/tasks/seed/posts.ts
+
 export default defineTask({
   meta: {
       name: "seed:posts",
@@ -18,6 +20,16 @@ export default defineTask({
     },
   run({ payload, context }) {
     const db = useDb();
+    const schema = useDbSchema();
+    const posts = [
+      {
+        title: "Hello World",
+        content: "This is a test post",
+      },
+      // TODO: Add more here
+    ];
+
+    db.insert(schema.posts).values(posts);
     // do seeding
     return { result: "Success" };
   },
