diff --git a/nuxt.config.ts b/nuxt.config.ts
index 8f98143..14895c5 100644
--- a/nuxt.config.ts
+++ b/nuxt.config.ts
@@ -38,6 +38,7 @@ export default defineNuxtConfig({
   },
   nitro: {
     experimental: {
+      tasks: true,
       openAPI: true,
     },
     openAPI: {
@@ -48,9 +49,6 @@ export default defineNuxtConfig({
         version: "1.0",
       },
     },
-    ignore: [
-      "**/*.test.{js,ts}",
-      "**/*.spec.{js,ts}",
-    ],
+    ignore: ["**/*.test.{js,ts}", "**/*.spec.{js,ts}"],
   },
-});
\ No newline at end of file
+});
