diff --git a/src/components/Contact/Contact.jsx b/src/components/Contact/Contact.jsx
index 797a023..77a3005 100644
--- a/src/components/Contact/Contact.jsx
+++ b/src/components/Contact/Contact.jsx
@@ -32,7 +32,7 @@ const Contact = () => {
 					)}
 				</span>
 			</p>
-			<div className="contact-link contact-item">
+			<div className="contact-item">
 				<LinkedInSVG />
 				<a
 					href={LINKEDIN_URL}
