diff --git a/.prettierrc b/.prettierrc
index 3d83cfb..d507638 100644
--- a/.prettierrc
+++ b/.prettierrc
@@ -1,8 +1,8 @@
 {
-    "semi": true,
-    "singleQuote": true,
-    "tabWidth": 2,
-    "trailingComma": "es5",
-    "printWidth": 100,
-    "arrowParens": "avoid"
-  }
\ No newline at end of file
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "es5",
+  "printWidth": 100,
+  "arrowParens": "avoid"
+}
