diff --git a/src/lib/firebase.ts b/src/lib/firebase.ts
index 9c7f1b2..d7261b4 100644
--- a/src/lib/firebase.ts
+++ b/src/lib/firebase.ts
@@ -7,7 +7,7 @@ const firebaseConfig = {
   projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
   storageBucket: process.env.NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET,
   messagingSenderId: process.env.NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID,
-  appId: process.env.NEXT_PUBLIC_FIREBASE_APP_ID
+  appId: process.env.NEXT_PUBLIC_FIREBASE_APP_ID,
 };
 
 // Initialize Firebase
@@ -16,4 +16,4 @@ const app = getApps().length === 0 ? initializeApp(firebaseConfig) : getApps()[0
 // Initialize Auth
 const auth = getAuth(app);
 
-export { auth }; 
\ No newline at end of file
+export { auth };
