diff --git a/src/lib/utils/date.ts b/src/lib/utils/date.ts
index fb264cb..d676493 100644
--- a/src/lib/utils/date.ts
+++ b/src/lib/utils/date.ts
@@ -2,6 +2,6 @@ export const formatDate = (dateString: string): string => {
   return new Date(dateString).toLocaleDateString('nl-NL', {
     day: 'numeric',
     month: 'long',
-    year: 'numeric'
+    year: 'numeric',
   });
-}; 
\ No newline at end of file
+};
