diff --git a/src/test/setup.ts b/src/test/setup.ts
index 94d0462..2602fff 100644
--- a/src/test/setup.ts
+++ b/src/test/setup.ts
@@ -3,6 +3,6 @@ import '@testing-library/jest-dom';
 // Mock voor navigator.clipboard
 Object.assign(navigator, {
   clipboard: {
-    writeText: jest.fn().mockImplementation(() => Promise.resolve())
-  }
-}); 
\ No newline at end of file
+    writeText: jest.fn().mockImplementation(() => Promise.resolve()),
+  },
+});
