diff --git a/game/tictactoe.go b/game/tictactoe.go
index c41e1df..19169f8 100644
--- a/game/tictactoe.go
+++ b/game/tictactoe.go
@@ -1,17 +1,13 @@
 package game
 
 type Game struct {
-	board []string
+	board [9]string
 	text  string
 }
 
-func (g *Game) Board() interface{} {
-	return g.board
-}
-
 func NewGame() *Game {
 	return &Game{
-		board: make([]string, 9),
+		board: [9]string{"", "", "", "", "", "", "", "", ""},
 		text:  "hello",
 	}
 }
@@ -19,3 +15,7 @@ func NewGame() *Game {
 func (g *Game) Text() string {
 	return g.text
 }
+
+func (g *Game) Board() [9]string {
+	return g.board
+}
