diff --git a/client/src/components/PlayerCharacter.vue b/client/src/components/PlayerCharacter.vue
index 5693f28..b14ff89 100644
--- a/client/src/components/PlayerCharacter.vue
+++ b/client/src/components/PlayerCharacter.vue
@@ -367,6 +367,13 @@ watch(() => heldItemId.value, (newValue) => {
     gameStore.emitEvent('clear-hint');
   }
 }, { immediate: true });
+
+// Watch for changes to interaction locked state
+watch(() => gameStore.interactionLocked, (newValue) => {
+  if (newValue) {
+    clearPressedKeys();
+  }
+});
 </script>
 
 <template>
