diff --git a/lib/src/screens/user_details/user_details_viewmodel.graphql b/lib/src/screens/user_details/user_details_viewmodel.graphql
new file mode 100644
index 0000000..ae72c58
--- /dev/null
+++ b/lib/src/screens/user_details/user_details_viewmodel.graphql
@@ -0,0 +1,22 @@
+#import "../../widgets/user_profile/user_profile.graphql"
+#import "../../widgets/repository_card/repository_card.graphql"
+
+query GetUserDetails($login: String!) {
+  user(login: $login) {
+    ...UserProfileFragment
+  }
+}
+
+query GetUserRepositories($login: String!, $first: Int!, $after: String) {
+  user(login: $login) {
+    repositories(first: $first, after: $after, orderBy: {field: UPDATED_AT, direction: DESC}) {
+      nodes {
+        ...RepositoryCardFragment
+      }
+      pageInfo {
+        hasNextPage
+        endCursor
+      }
+    }
+  }
+}
\ No newline at end of file
