diff --git a/src/app/game/game_config.py b/src/app/game/game_config.py
index 93ae4e7..eb5e0ab 100644
--- a/src/app/game/game_config.py
+++ b/src/app/game/game_config.py
@@ -98,4 +98,10 @@ def load_game_config(data_file: Path) -> GameConfig:
         pokemons=pokemons,
         sections=sections,
         quizzes_by_id=quizzes_by_id
-    ) 
\ No newline at end of file
+    )
+
+
+def load_equation_difficulties(file_path):
+    with open(file_path) as f:
+        difficulties = json.load(f)
+    return difficulties
