diff --git a/vite.config.ts b/vite.config.ts
index d6dffa8..f90d283 100644
--- a/vite.config.ts
+++ b/vite.config.ts
@@ -7,14 +7,14 @@ import tsconfigPaths from 'vite-tsconfig-paths';
 // https://vite.dev/config/
 export default defineConfig({
   plugins: [
-    react(),
-    tailwindcss(),
     tanstackRouter({
       target: 'react',
       autoCodeSplitting: true,
       routesDirectory: './src/app/routes',
       generatedRouteTree: './src/shared/utils/router/routeTree.gen.ts',
     }),
+    react(),
+    tailwindcss(),
     tsconfigPaths(),
   ],
 });
